<!DOCTYPE html><!--[if IE 9]><html class="lt-ie10" lang="en" > <![endif]--><!--[if IE 10]><html class="ie10" lang="en" > <![endif]--><html class="no-js" lang="en">
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>nCounter Advanced Analysis Report</title>
    <meta name="author" content="NanoString Technologies" />
    <link rel="stylesheet" href="resources/css/normalize.css" />
    <link rel="stylesheet" href="resources/css/foundation.css" />
    <link rel="stylesheet" href="resources/css/jquery.dataTables.min.css" />
    <link rel="stylesheet" href="resources/css/jquery-ui.css" />
    <link rel="stylesheet" href="resources/css/theme.css" />
    <link rel="stylesheet" href="resources/css/report.css" />
    <link rel="stylesheet" href="resources/css/heatmap.css" />
    <script src="resources/js/modernizr.js"> </script>
    <script src="resources/js/jquery.js"> </script>
    <script src="resources/js/jquery-ui.min.js"> </script>
    <script src="resources/js/fastclick.js"> </script>
    <script src="resources/js/foundation.js"> </script>
    <script src="resources/js/foundation.tab.js"> </script>
    <script src="resources/js/foundation.topbar.js"> </script>
    <script src="resources/js/foundation.dropdown.js"> </script>
    <script src="resources/js/jquery.dataTables.min.js"> </script>
    <script src="resources/js/report.js"> </script>
    <script src="resources/inc/share.js"> </script>
    <script src="resources/js/d3.min.js"> </script>
    <script src="resources/js/heatmap.js"> </script>
    <script src="resources/js/helpers.js"> </script>
    <script src="resources/js/advanced.analysis.js"> </script>
    <link rel="shortcut icon" type="Image/x-icon" href="resources/img/nanostring_icon.png" />
    <meta class="foundation-data-attribute-namespace" />
    <meta class="foundation-mq-xxlarge" />
    <meta class="foundation-mq-xlarge" />
    <meta class="foundation-mq-large" />
    <meta class="foundation-mq-medium" />
    <meta class="foundation-mq-small" />
    <meta class="foundation-data-attribute-namespace" />
    <meta class="foundation-mq-xxlarge" />
    <meta class="foundation-mq-xlarge" />
    <meta class="foundation-mq-large" />
    <meta class="foundation-mq-medium" />
    <meta class="foundation-mq-small" />
    <meta class="foundation-mq-topbar" />
    <meta class="foundation-mq-topbar" />
    <meta class="foundation-mq-topbar" />
  </head>
  <body>
    <nav class="top-bar" data-topbar="" data-options="is_hover: false" role="navigation">
      <ul class="title-area">
        <li class="name">
          <h1>
            <a href="http://www.nanostring.com">
              <img src="resources/img/logo_nanostring_white_Flat_189x40.png" />
            </a>
          </h1>
        </li>
      </ul>
      <section class="top-bar-section">
        <ul class="right">
      
        <li class="active"><a href="javascript:ShowContent('ULFu_7-xiaLT4hE9u9s3cQw')">Overview</a></li>
        <li><a href="javascript:ShowContent('Ud8oJD0a0rBs_stlncl5cxw')">Diff Expr</a></li>
        <li><a href="javascript:ShowContent('U9YhWvqi0JdlimyJ-bFsxwA')">PathView</a></li>
        <li><a href="javascript:ShowContent('UqWuHjesj-_-CPzfxWVp58Q')">Cell Type Profiling</a></li>
        <li><a href="javascript:ShowContent('UgJSZ36tfOj9Lv9WE-UAU3Q')">Analysis Parameters</a></li>
      
    <li class="share-report"><a href="AdvAnalysisReport.zip">Share</a></li></ul>
      </section>
    </nav>
    <div class="row">
      
        
          
            <div id="ULFu_7-xiaLT4hE9u9s3cQw" class="panelContent" style="display:block"><div class="row"><div class="panel"><p>This module generates a series of high level plots that describe the data overall and may be useful for identifying anomalous data and/or covariates.</p></div></div><div class="row"><div class="small-12 columns">
              
                <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Gene sets:</p>
                  
                    <li class="tab-title active"><a href="#U5WP8n1ap0gvp2mvI5hu36w">Summary</a></li>
                  
                </ul></div><div class="tabs-content">
                  
                    
                      
                        <div id="U5WP8n1ap0gvp2mvI5hu36w" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                          
                            <dl class="button-group radius"><li><a class="small button active" href="#" data-dropdown="drop-Ue4fBTGUi0K6a3UIZy2EXWw" title="Heatmaps">Heatmaps</a></li><li><a class="small button" href="#" data-dropdown="drop-UGcyv357jNP2Y8IH9RjHRrw" title="PCA">PCA</a></li><li><a class="small button" href="#" data-dropdown="drop-U23vvzqJuW63fGNPtgvG0eg" title="Study design">Study design</a></li><li><a class="small button" href="#" data-dropdown="drop-URDO2JI2a4XDN-Zk8osGCaQ" title="Other QC">Other QC</a></li></dl><ul id="drop-Ue4fBTGUi0K6a3UIZy2EXWw" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                              
                                <li class="active"><a menuindex="Heatmaps|Heatmaps" href="#UoEJXAnpaQcc4Vo5y6qasrA">Heatmaps</a></li>
                              
                            </ul><ul id="drop-UGcyv357jNP2Y8IH9RjHRrw" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                              
                                <li><a menuindex="PCA|Patient" href="#Uk3th4u_woFrwO0xPIh_9uA">Patient</a></li>
                                <li><a menuindex="PCA|Disease" href="#U29GjaBOGVAiFMTwbS72rSg">Disease</a></li>
                                <li><a menuindex="PCA|Drug" href="#UYGDaDbOQ6e7FHm9YuKNRnA">Drug</a></li>
                                <li><a menuindex="PCA|Dosage" href="#UncpAawtKB_Ph4DYkRD6mNQ">Dosage</a></li>
                                <li><a menuindex="PCA|Day" href="#UpsZK6r32lTF27lAYT620pg">Day</a></li>
                              
                            </ul><ul id="drop-U23vvzqJuW63fGNPtgvG0eg" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                              
                                <li><a menuindex="Study design|Covariates" href="#U0O4EAIH6GrmXFZe1nr0Cdg">Covariates</a></li>
                              
                            </ul><ul id="drop-URDO2JI2a4XDN-Zk8osGCaQ" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                              
                                <li><a menuindex="Other QC|Other QC" href="#UdLJ5qOajDxW5Yg9q2AQDsQ">Other QC</a></li>
                              
                            </ul><div class="tabs-content">
                            
                              
                                
                                  
                                    <div id="UoEJXAnpaQcc4Vo5y6qasrA" class="content active"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Heatmap of Raw Data<br /><a href="#" onclick="openPopup('UTcPLX0AhDFWn7XRG6-j1tQ_UQh1YWZvRyUX_kmCGIztRmQ');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/QC/above background detection call.csv"><span class="label round">Download detected/undetected calls data</span></a></b></p><a href="#" onclick="openInteractiveHeatmapPopup('UQh1YWZvRyUX_kmCGIztRmQ');return false;"><div class="small-9 columns"><image src="./results/QC/raw data heatmap.png" /></div><div class="small-3 columns"><image src="./results/QC/raw data heatmap - color key.png" /></div></a><div id="UQh1YWZvRyUX_kmCGIztRmQ" class="heatmap_pop_up"><div class="image_pop_up_content small-12 columns"><div id="ih-div-UQh1YWZvRyUX_kmCGIztRmQ" /></div></div><div id="UTcPLX0AhDFWn7XRG6-j1tQ_UQh1YWZvRyUX_kmCGIztRmQ" class="annotation_pop_up"><h5>Heatmap of Raw Data</h5><p>Heatmap of the raw counts. The plot is meant to provide an overview of how robust the raw expression levels are across samples and gene sets. Datasets that entirely lack higher level expressions (e.g. counts &gt; 100) may indicate experimental failure or low input. The detected/undetected calls links to a .csv file stating whether each probe is above background, with 0/1 indicating below/above background. If the user has not specified a detection threshold, probes are called detected if they have more than double the counts of the median negative control.</p></div></li>
                                        <li><p><b>Heatmap of All Data<br /><a href="#" onclick="openPopup('UB9DF-SbN2-tvUgxUJw-L2g_UyySR3RU_9hQ_hWdumMiueg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openInteractiveHeatmapPopup('UyySR3RU_9hQ_hWdumMiueg');return false;"><div class="small-9 columns"><image src="./results/QC/heatmap of all genes X samples.png" /></div><div class="small-3 columns"><image src="./results/QC/normalized data heatmap - color key.png" /></div></a><div id="UyySR3RU_9hQ_hWdumMiueg" class="heatmap_pop_up"><div class="image_pop_up_content small-12 columns"><div id="ih-div-UyySR3RU_9hQ_hWdumMiueg" /></div></div><div id="UB9DF-SbN2-tvUgxUJw-L2g_UyySR3RU_9hQ_hWdumMiueg" class="annotation_pop_up"><h5>Heatmap of All Data</h5><p>Heatmap of the normalized data, scaled to give all genes equal variance, generated via unsupervised clustering. Orange indicates high expression; blue indicates low expression. This plot is meant to provide a high level exploratory view of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            
                            
                              
                                
                                  
                                    <div id="Uk3th4u_woFrwO0xPIh_9uA" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Principal Components of All Data<br /><a href="#" onclick="openPopup('URZEC0kb7QZVDEJbscGGcRA_UxMrornnvtkIhyXPLj0P0pg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UxMrornnvtkIhyXPLj0P0pg');return false;"><div class="small-9 columns"><image src="./results/QC/PCA colored by Patient.png" /></div><div class="small-3 columns"><image src="./results/color legend - Patient.png" /></div></a><div id="UxMrornnvtkIhyXPLj0P0pg" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/QC/PCA colored by Patient.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Patient.png" /></div></div><div id="URZEC0kb7QZVDEJbscGGcRA_UxMrornnvtkIhyXPLj0P0pg" class="annotation_pop_up"><h5>Principal Components of All Data</h5><p>Principal component analysis maps high-dimensional datasets onto a smaller number of highly informative dimensions. Here, the first four principal components of the gene expression data are plotted against each other and colored by the values of the selected covariate. This plot may be used to identify clusters in the data and to identify variables associated with prominent signal in the data. Variables that are associated with these leading principal components should be considered in downstream analyses.</p></div></li>
                                        <li><p><b>Outliers<br /><a href="#" onclick="openPopup('UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><table id="U09S07bqTn9DfcNQy1vhGpQ" class="display" cellspacing="0">
                                          <Content Count="1">
                                            <Annotation Id="UJY0Wwdf5LW_ibbwv7HAf3w" Title="Table identifying outliers in the first four PCs of the data." />
                                          </Content>
                                          <thead>
                                            <tr>
                                              <td />
                                              <td>Outlier on PC 1</td>
                                              <td>Outlier on PC 2</td>
                                              <td>Outlier on PC 3</td>
                                              <td>Outlier on PC 4</td>
                                            </tr>
                                          </thead>
                                          <tbody>
                                            <tr>
                                              <td>20210112_30102534301221-01_P0091-3 D0 media_06.RCC</td>
                                              <td>FALSE</td>
                                              <td>TRUE</td>
                                              <td>FALSE</td>
                                              <td>FALSE</td>
                                            </tr>
                                          </tbody>
                                        </table><div id="UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ" class="annotation_pop_up"><h5>Outliers</h5><p>Table identifying outliers in the first four PCs of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U29GjaBOGVAiFMTwbS72rSg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Principal Components of All Data<br /><a href="#" onclick="openPopup('URZEC0kb7QZVDEJbscGGcRA_UYvb-uTe7UNhGU4nKho6bTQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UYvb-uTe7UNhGU4nKho6bTQ');return false;"><div class="small-9 columns"><image src="./results/QC/PCA colored by Disease.png" /></div><div class="small-3 columns"><image src="./results/color legend - Disease.png" /></div></a><div id="UYvb-uTe7UNhGU4nKho6bTQ" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/QC/PCA colored by Disease.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Disease.png" /></div></div><div id="URZEC0kb7QZVDEJbscGGcRA_UYvb-uTe7UNhGU4nKho6bTQ" class="annotation_pop_up"><h5>Principal Components of All Data</h5><p>Principal component analysis maps high-dimensional datasets onto a smaller number of highly informative dimensions. Here, the first four principal components of the gene expression data are plotted against each other and colored by the values of the selected covariate. This plot may be used to identify clusters in the data and to identify variables associated with prominent signal in the data. Variables that are associated with these leading principal components should be considered in downstream analyses.</p></div></li>
                                        <li><p><b>Outliers<br /><a href="#" onclick="openPopup('UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><table id="U09S07bqTn9DfcNQy1vhGpQ" class="display" cellspacing="0">
                                          <Content Count="1">
                                            <Annotation Id="UJY0Wwdf5LW_ibbwv7HAf3w" Title="Table identifying outliers in the first four PCs of the data." />
                                          </Content>
                                          <thead>
                                            <tr>
                                              <td />
                                              <td>Outlier on PC 1</td>
                                              <td>Outlier on PC 2</td>
                                              <td>Outlier on PC 3</td>
                                              <td>Outlier on PC 4</td>
                                            </tr>
                                          </thead>
                                          <tbody>
                                            <tr>
                                              <td>20210112_30102534301221-01_P0091-3 D0 media_06.RCC</td>
                                              <td>FALSE</td>
                                              <td>TRUE</td>
                                              <td>FALSE</td>
                                              <td>FALSE</td>
                                            </tr>
                                          </tbody>
                                        </table><div id="UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ" class="annotation_pop_up"><h5>Outliers</h5><p>Table identifying outliers in the first four PCs of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UYGDaDbOQ6e7FHm9YuKNRnA" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Principal Components of All Data<br /><a href="#" onclick="openPopup('URZEC0kb7QZVDEJbscGGcRA_UypdXjHIUYXTnhyeiLSmnaw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UypdXjHIUYXTnhyeiLSmnaw');return false;"><div class="small-9 columns"><image src="./results/QC/PCA colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UypdXjHIUYXTnhyeiLSmnaw" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/QC/PCA colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="URZEC0kb7QZVDEJbscGGcRA_UypdXjHIUYXTnhyeiLSmnaw" class="annotation_pop_up"><h5>Principal Components of All Data</h5><p>Principal component analysis maps high-dimensional datasets onto a smaller number of highly informative dimensions. Here, the first four principal components of the gene expression data are plotted against each other and colored by the values of the selected covariate. This plot may be used to identify clusters in the data and to identify variables associated with prominent signal in the data. Variables that are associated with these leading principal components should be considered in downstream analyses.</p></div></li>
                                        <li><p><b>Outliers<br /><a href="#" onclick="openPopup('UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><table id="U09S07bqTn9DfcNQy1vhGpQ" class="display" cellspacing="0">
                                          <Content Count="1">
                                            <Annotation Id="UJY0Wwdf5LW_ibbwv7HAf3w" Title="Table identifying outliers in the first four PCs of the data." />
                                          </Content>
                                          <thead>
                                            <tr>
                                              <td />
                                              <td>Outlier on PC 1</td>
                                              <td>Outlier on PC 2</td>
                                              <td>Outlier on PC 3</td>
                                              <td>Outlier on PC 4</td>
                                            </tr>
                                          </thead>
                                          <tbody>
                                            <tr>
                                              <td>20210112_30102534301221-01_P0091-3 D0 media_06.RCC</td>
                                              <td>FALSE</td>
                                              <td>TRUE</td>
                                              <td>FALSE</td>
                                              <td>FALSE</td>
                                            </tr>
                                          </tbody>
                                        </table><div id="UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ" class="annotation_pop_up"><h5>Outliers</h5><p>Table identifying outliers in the first four PCs of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UncpAawtKB_Ph4DYkRD6mNQ" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Principal Components of All Data<br /><a href="#" onclick="openPopup('URZEC0kb7QZVDEJbscGGcRA_UtMoCJw48y1tLCfzjQh6GNA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UtMoCJw48y1tLCfzjQh6GNA');return false;"><div class="small-9 columns"><image src="./results/QC/PCA colored by Dosage.png" /></div><div class="small-3 columns"><image src="./results/color legend - Dosage.png" /></div></a><div id="UtMoCJw48y1tLCfzjQh6GNA" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/QC/PCA colored by Dosage.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Dosage.png" /></div></div><div id="URZEC0kb7QZVDEJbscGGcRA_UtMoCJw48y1tLCfzjQh6GNA" class="annotation_pop_up"><h5>Principal Components of All Data</h5><p>Principal component analysis maps high-dimensional datasets onto a smaller number of highly informative dimensions. Here, the first four principal components of the gene expression data are plotted against each other and colored by the values of the selected covariate. This plot may be used to identify clusters in the data and to identify variables associated with prominent signal in the data. Variables that are associated with these leading principal components should be considered in downstream analyses.</p></div></li>
                                        <li><p><b>Outliers<br /><a href="#" onclick="openPopup('UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><table id="U09S07bqTn9DfcNQy1vhGpQ" class="display" cellspacing="0">
                                          <Content Count="1">
                                            <Annotation Id="UJY0Wwdf5LW_ibbwv7HAf3w" Title="Table identifying outliers in the first four PCs of the data." />
                                          </Content>
                                          <thead>
                                            <tr>
                                              <td />
                                              <td>Outlier on PC 1</td>
                                              <td>Outlier on PC 2</td>
                                              <td>Outlier on PC 3</td>
                                              <td>Outlier on PC 4</td>
                                            </tr>
                                          </thead>
                                          <tbody>
                                            <tr>
                                              <td>20210112_30102534301221-01_P0091-3 D0 media_06.RCC</td>
                                              <td>FALSE</td>
                                              <td>TRUE</td>
                                              <td>FALSE</td>
                                              <td>FALSE</td>
                                            </tr>
                                          </tbody>
                                        </table><div id="UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ" class="annotation_pop_up"><h5>Outliers</h5><p>Table identifying outliers in the first four PCs of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UpsZK6r32lTF27lAYT620pg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Principal Components of All Data<br /><a href="#" onclick="openPopup('URZEC0kb7QZVDEJbscGGcRA_UnHCFdrqdgsGNMMcDg0vqPA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UnHCFdrqdgsGNMMcDg0vqPA');return false;"><div class="small-9 columns"><image src="./results/QC/PCA colored by Day.png" /></div><div class="small-3 columns"><image src="./results/color legend - Day.png" /></div></a><div id="UnHCFdrqdgsGNMMcDg0vqPA" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/QC/PCA colored by Day.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Day.png" /></div></div><div id="URZEC0kb7QZVDEJbscGGcRA_UnHCFdrqdgsGNMMcDg0vqPA" class="annotation_pop_up"><h5>Principal Components of All Data</h5><p>Principal component analysis maps high-dimensional datasets onto a smaller number of highly informative dimensions. Here, the first four principal components of the gene expression data are plotted against each other and colored by the values of the selected covariate. This plot may be used to identify clusters in the data and to identify variables associated with prominent signal in the data. Variables that are associated with these leading principal components should be considered in downstream analyses.</p></div></li>
                                        <li><p><b>Outliers<br /><a href="#" onclick="openPopup('UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><table id="U09S07bqTn9DfcNQy1vhGpQ" class="display" cellspacing="0">
                                          <Content Count="1">
                                            <Annotation Id="UJY0Wwdf5LW_ibbwv7HAf3w" Title="Table identifying outliers in the first four PCs of the data." />
                                          </Content>
                                          <thead>
                                            <tr>
                                              <td />
                                              <td>Outlier on PC 1</td>
                                              <td>Outlier on PC 2</td>
                                              <td>Outlier on PC 3</td>
                                              <td>Outlier on PC 4</td>
                                            </tr>
                                          </thead>
                                          <tbody>
                                            <tr>
                                              <td>20210112_30102534301221-01_P0091-3 D0 media_06.RCC</td>
                                              <td>FALSE</td>
                                              <td>TRUE</td>
                                              <td>FALSE</td>
                                              <td>FALSE</td>
                                            </tr>
                                          </tbody>
                                        </table><div id="UJY0Wwdf5LW_ibbwv7HAf3w_U09S07bqTn9DfcNQy1vhGpQ" class="annotation_pop_up"><h5>Outliers</h5><p>Table identifying outliers in the first four PCs of the data.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            
                            
                              
                                
                                  
                                    <div id="U0O4EAIH6GrmXFZe1nr0Cdg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UJfMm6o4k6-ZMU0Z0ANslQQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UJfMm6o4k6-ZMU0Z0ANslQQ');return false;"><div class="small-12 columns"><image src="./results/QC/univariate plot of covariates - Day.png" /></div></a><div id="UJfMm6o4k6-ZMU0Z0ANslQQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/univariate plot of covariates - Day.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UJfMm6o4k6-ZMU0Z0ANslQQ" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UH4nyQ8XvMmgJxa1tPwmvHA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UH4nyQ8XvMmgJxa1tPwmvHA');return false;"><div class="small-12 columns"><image src="./results/QC/univariate plot of covariates - Disease.png" /></div></a><div id="UH4nyQ8XvMmgJxa1tPwmvHA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/univariate plot of covariates - Disease.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UH4nyQ8XvMmgJxa1tPwmvHA" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UDZdXEoAE11q-FJKmMyC1_w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UDZdXEoAE11q-FJKmMyC1_w');return false;"><div class="small-12 columns"><image src="./results/QC/univariate plot of covariates - Dosage.png" /></div></a><div id="UDZdXEoAE11q-FJKmMyC1_w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/univariate plot of covariates - Dosage.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UDZdXEoAE11q-FJKmMyC1_w" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UlAnku8QQGuG9TxWEziC9xQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UlAnku8QQGuG9TxWEziC9xQ');return false;"><div class="small-12 columns"><image src="./results/QC/univariate plot of covariates - Drug.png" /></div></a><div id="UlAnku8QQGuG9TxWEziC9xQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/univariate plot of covariates - Drug.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UlAnku8QQGuG9TxWEziC9xQ" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_U3jcsTuwt6HsO1j7fQ5YwGw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U3jcsTuwt6HsO1j7fQ5YwGw');return false;"><div class="small-12 columns"><image src="./results/QC/univariate plot of covariates - Patient.png" /></div></a><div id="U3jcsTuwt6HsO1j7fQ5YwGw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/univariate plot of covariates - Patient.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_U3jcsTuwt6HsO1j7fQ5YwGw" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UOcD-tcsyswn6w8Cd9MJLJQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UOcD-tcsyswn6w8Cd9MJLJQ');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Disease vs Day.png" /></div></a><div id="UOcD-tcsyswn6w8Cd9MJLJQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Disease vs Day.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UOcD-tcsyswn6w8Cd9MJLJQ" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UiXC_47NgCCN3RF6qUYUJtg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UiXC_47NgCCN3RF6qUYUJtg');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Disease vs Dosage.png" /></div></a><div id="UiXC_47NgCCN3RF6qUYUJtg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Disease vs Dosage.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UiXC_47NgCCN3RF6qUYUJtg" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_UYnY3H0kuEX1xt7fpDrCkMg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UYnY3H0kuEX1xt7fpDrCkMg');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Disease vs Drug.png" /></div></a><div id="UYnY3H0kuEX1xt7fpDrCkMg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Disease vs Drug.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_UYnY3H0kuEX1xt7fpDrCkMg" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_U7qtqkR4tc0klPTkkm4OdOg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U7qtqkR4tc0klPTkkm4OdOg');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Dosage vs Day.png" /></div></a><div id="U7qtqkR4tc0klPTkkm4OdOg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Dosage vs Day.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_U7qtqkR4tc0klPTkkm4OdOg" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URxyLKEr74V9EjMexkw5pTQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URxyLKEr74V9EjMexkw5pTQ');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Drug vs Day.png" /></div></a><div id="URxyLKEr74V9EjMexkw5pTQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Drug vs Day.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URxyLKEr74V9EjMexkw5pTQ" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Drug vs Dosage.png" /></div></a><div id="URfJPC7SnhKLVb4QyVr-Z_Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Drug vs Dosage.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Patient vs Day.png" /></div></a><div id="URfJPC7SnhKLVb4QyVr-Z_Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Patient vs Day.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Patient vs Disease.png" /></div></a><div id="URfJPC7SnhKLVb4QyVr-Z_Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Patient vs Disease.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Patient vs Dosage.png" /></div></a><div id="URfJPC7SnhKLVb4QyVr-Z_Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Patient vs Dosage.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                        <li><p><b><br /><a href="#" onclick="openPopup('UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('URfJPC7SnhKLVb4QyVr-Z_Q');return false;"><div class="small-12 columns"><image src="./results/QC/pairs plot of covariates - Patient vs Drug.png" /></div></a><div id="URfJPC7SnhKLVb4QyVr-Z_Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/pairs plot of covariates - Patient vs Drug.png" style="width: 100%;" /></div></div><div id="UTO5fqmQLelAKMcrSCUf8Bg_URfJPC7SnhKLVb4QyVr-Z_Q" class="annotation_pop_up"><h5></h5><p>Pairwise comparisons of all covariates in the analysis. The type of plot is dependent on the types of variables compared; A categorical vs. categorical covariate plot is shown as a bar chart of counts (Y axis). Continuous vs. categorical covariates generate a boxplot with whiskers denoting 1.5 IQR. Continuous vs. continuous covariates are compared via a scatter plot. Variables that are correlated with a biological variable of interest are potential confounders that may influence downstream analyses. Additionally, bar plots and histograms show the distributions of categorical and continuous variables, respectively.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            
                            
                              
                                
                                  
                                    <div id="UdLJ5qOajDxW5Yg9q2AQDsQ" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Variance vs. Mean normalized signal plot across all targets/probes<br /><a href="#" onclick="openPopup('Uia-4G0f8ZajKI2LEHseQGA_UV_rAwzIfsyQnOHSDocfmRA');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/QC/QC module - Mean.Var.normalized.counts.csv"><span class="label round">Mean and Variance statistics across all genes</span></a></b></p><a href="#" onclick="openImagePopup('UV_rAwzIfsyQnOHSDocfmRA');return false;"><div class="small-12 columns"><image src="./results/QC/mean vs var for all genes.png" /></div></a><div id="UV_rAwzIfsyQnOHSDocfmRA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/mean vs var for all genes.png" style="width: 100%;" /></div></div><div id="Uia-4G0f8ZajKI2LEHseQGA_UV_rAwzIfsyQnOHSDocfmRA" class="annotation_pop_up"><h5>Variance vs. Mean normalized signal plot across all targets/probes</h5><p>Each gene's variance in the log-scaled, normalized data is plotted against its mean value across all samples. Highly variable genes are indicated by gene name. Housekeeping genes are color coded according to their use in (or omission from) normalization.</p></div></li>
                                        <li><p><b>p-value distribution plots<br /><a href="#" onclick="openPopup('UZivnbxg7treB5U6bs1M4Lg_UlGdrykQ_XClUjIvsKudbjg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UlGdrykQ_XClUjIvsKudbjg');return false;"><div class="small-12 columns"><image src="./results/QC/p-value histograms.png" /></div></a><div id="UlGdrykQ_XClUjIvsKudbjg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/QC/p-value histograms.png" style="width: 100%;" /></div></div><div id="UZivnbxg7treB5U6bs1M4Lg_UlGdrykQ_XClUjIvsKudbjg" class="annotation_pop_up"><h5>p-value distribution plots</h5><p>For each covariate included in the analysis, a histogram of p-values testing each gene's univariate association with the chosen covariate is displayed. Covariates with largely flat histograms have minimal association with gene expression; covariates with histograms with significantly more mass on the left are either associated with the expression of many genes or are confounded with a covariate that is associated with the expression. Low p-values indicate strong evidence for an association.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            
                          </div>
                            
                            
                            
                          
                        </ul></div></div>
                      
                    
                  
                </div>
              
            </div></div></div>
          
        
        
          
            <div id="Ud8oJD0a0rBs_stlncl5cxw" class="panelContent" style="display:none"><div class="row"><div class="panel"><p>Genes are tested for differential expression in response to each selected covariate. For each gene, a single linear 
  regression is fit using all selected covariates to predict expression. This approach eliminates confounding due to measured
  covariates and isolates the independent association of each covariate with gene expression, measuring each variable's
  association with a gene after holding all other variables constant.</p></div></div><div class="row"><div class="panel"><ul class="small-block-grid-1">
              
                <dl class="sub-nav" data-tab=""><dt>Drug:</dt>
                  
                    <li class="active"><a menuindex="Drug|Drug: differential expression in IFNy vs. baseline of Media" href="#U8MiG_WqCLyMjDB0uxcd15A">Drug: differential expression in IFNy vs. baseline of Media</a></li>
                    <li><a menuindex="Drug|Drug: differential expression in IgG vs. baseline of Media" href="#UImlUPt-OzhuDCQwu7_f-tg">Drug: differential expression in IgG vs. baseline of Media</a></li>
                    <li><a menuindex="Drug|Drug: differential expression in NMC2 vs. baseline of Media" href="#UP734xI-HVQYhnax6P4UkJw">Drug: differential expression in NMC2 vs. baseline of Media</a></li>
                  
                </dl><div class="tabs-content">
                  
                    
                      
                        <div id="U8MiG_WqCLyMjDB0uxcd15A" class="content active"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                          
                            <li><p><b>Volcano Plot: Drug: IFNy vs.Media<br /><a href="#" onclick="openPopup('U13nWZ4nPj-rwES2iGGSYNA_UrcK6lfZDoV5vbPKZsaqjWA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UrcK6lfZDoV5vbPKZsaqjWA');return false;"><div class="small-12 columns"><image src="./results/DE/volcano plotDrugIFNy.png" /></div></a><div id="UrcK6lfZDoV5vbPKZsaqjWA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/DE/volcano plotDrugIFNy.png" style="width: 100%;" /></div></div><div id="U13nWZ4nPj-rwES2iGGSYNA_UrcK6lfZDoV5vbPKZsaqjWA" class="annotation_pop_up"><h5>Volcano Plot: Drug: IFNy vs.Media</h5><p>Volcano plot displaying each gene's -log10(p-value) and log2 fold change with the selected covariate. Highly statistically significant genes fall at the top of the plot above the horizontal lines, and highly differentially expressed genes fall to either side. Horizontal lines indicate various False Discovery Rate (FDR) thresholds or p-value thresholds if there is no adjustment to the p-values. Genes are colored if the resulting p-value is below the given FDR or p-value threshold. The 40 most statistically significant genes are labeled in the plot.</p></div></li>
                            <li><p><b>DE Results: Drug: IFNy vs.Media<br /><a href="#" onclick="openPopup('UK6pdMEHKbJBk9cwqKLAbwg_UKZeCryr1hHv4Wh6W8cAWyA');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/DE/DE Results - DrugIFNy.csv"><span class="label round">Download CSV Data</span></a></b></p><table id="UKZeCryr1hHv4Wh6W8cAWyA" class="display" cellspacing="0">
                              <Content Count="2">
                                <Annotation Id="UK6pdMEHKbJBk9cwqKLAbwg" Title="Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function." />
                                <FileDownloadLink Id="Ud_E7G1rYKJXWB5BNTmycEw" uri="./results/DE/DE Results - DrugIFNy.csv" Title="Download CSV Data" />
                              </Content>
                              <thead>
                                <tr>
                                  <td />
                                  <td>Log2 fold change</td>
                                  <td>std error (log2)</td>
                                  <td>Lower confidence limit (log2)</td>
                                  <td>Upper confidence limit (log2)</td>
                                  <td>Linear fold change</td>
                                  <td>Lower confidence limit (linear)</td>
                                  <td>Upper confidence limit (linear)</td>
                                  <td>P-value</td>
                                  <td>BY.p.value</td>
                                  <td>method</td>
                                  <td>probe.ID</td>
                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td>ATF3-mRNA</td>
                                  <td class="numeric_cell">-3.33</td>
                                  <td class="numeric_cell">0.334</td>
                                  <td class="numeric_cell">-3.98</td>
                                  <td class="numeric_cell">-2.67</td>
                                  <td class="numeric_cell">0.0997</td>
                                  <td class="numeric_cell">0.0633</td>
                                  <td class="numeric_cell">0.157</td>
                                  <td class="numeric_cell">2.37e-10</td>
                                  <td class="numeric_cell">2e-07</td>
                                  <td>lm.nb</td>
                                  <td>NM_001674.3:1487</td>
                                </tr>
                                <tr>
                                  <td>NFKBIA-mRNA</td>
                                  <td class="numeric_cell">-5</td>
                                  <td class="numeric_cell">0.678</td>
                                  <td class="numeric_cell">-6.33</td>
                                  <td class="numeric_cell">-3.67</td>
                                  <td class="numeric_cell">0.0313</td>
                                  <td class="numeric_cell">0.0125</td>
                                  <td class="numeric_cell">0.0787</td>
                                  <td class="numeric_cell">7.94e-08</td>
                                  <td class="numeric_cell">3.36e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_020529.2:1361</td>
                                </tr>
                                <tr>
                                  <td>CXCL14-mRNA</td>
                                  <td class="numeric_cell">-2.62</td>
                                  <td class="numeric_cell">0.371</td>
                                  <td class="numeric_cell">-3.35</td>
                                  <td class="numeric_cell">-1.89</td>
                                  <td class="numeric_cell">0.163</td>
                                  <td class="numeric_cell">0.0982</td>
                                  <td class="numeric_cell">0.269</td>
                                  <td class="numeric_cell">1.65e-07</td>
                                  <td class="numeric_cell">4.48e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_004887.4:990</td>
                                </tr>
                                <tr>
                                  <td>SGK1-mRNA</td>
                                  <td class="numeric_cell">-2.25</td>
                                  <td class="numeric_cell">0.322</td>
                                  <td class="numeric_cell">-2.88</td>
                                  <td class="numeric_cell">-1.62</td>
                                  <td class="numeric_cell">0.211</td>
                                  <td class="numeric_cell">0.136</td>
                                  <td class="numeric_cell">0.326</td>
                                  <td class="numeric_cell">2.12e-07</td>
                                  <td class="numeric_cell">4.48e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_005627.3:1621</td>
                                </tr>
                                <tr>
                                  <td>EGR1-mRNA</td>
                                  <td class="numeric_cell">-2.54</td>
                                  <td class="numeric_cell">0.39</td>
                                  <td class="numeric_cell">-3.31</td>
                                  <td class="numeric_cell">-1.78</td>
                                  <td class="numeric_cell">0.172</td>
                                  <td class="numeric_cell">0.101</td>
                                  <td class="numeric_cell">0.292</td>
                                  <td class="numeric_cell">6.57e-07</td>
                                  <td class="numeric_cell">0.000111</td>
                                  <td>lm.nb</td>
                                  <td>NM_001964.2:2034</td>
                                </tr>
                                <tr>
                                  <td>DUSP1-mRNA</td>
                                  <td class="numeric_cell">-4.77</td>
                                  <td class="numeric_cell">0.755</td>
                                  <td class="numeric_cell">-6.25</td>
                                  <td class="numeric_cell">-3.29</td>
                                  <td class="numeric_cell">0.0367</td>
                                  <td class="numeric_cell">0.0132</td>
                                  <td class="numeric_cell">0.102</td>
                                  <td class="numeric_cell">1.09e-06</td>
                                  <td class="numeric_cell">0.000153</td>
                                  <td>lm.nb</td>
                                  <td>NM_004417.3:1428</td>
                                </tr>
                                <tr>
                                  <td>THBS1-mRNA</td>
                                  <td class="numeric_cell">-4.51</td>
                                  <td class="numeric_cell">0.777</td>
                                  <td class="numeric_cell">-6.03</td>
                                  <td class="numeric_cell">-2.98</td>
                                  <td class="numeric_cell">0.044</td>
                                  <td class="numeric_cell">0.0153</td>
                                  <td class="numeric_cell">0.126</td>
                                  <td class="numeric_cell">4.12e-06</td>
                                  <td class="numeric_cell">0.000497</td>
                                  <td>lm.nb</td>
                                  <td>NM_003246.2:3816</td>
                                </tr>
                                <tr>
                                  <td>TGFB1-mRNA</td>
                                  <td class="numeric_cell">-2.14</td>
                                  <td class="numeric_cell">0.403</td>
                                  <td class="numeric_cell">-2.93</td>
                                  <td class="numeric_cell">-1.36</td>
                                  <td class="numeric_cell">0.226</td>
                                  <td class="numeric_cell">0.131</td>
                                  <td class="numeric_cell">0.391</td>
                                  <td class="numeric_cell">1.43e-05</td>
                                  <td class="numeric_cell">0.00151</td>
                                  <td>lm.nb</td>
                                  <td>NM_000660.4:1473</td>
                                </tr>
                                <tr>
                                  <td>CEBPB-mRNA</td>
                                  <td class="numeric_cell">-2.01</td>
                                  <td class="numeric_cell">0.39</td>
                                  <td class="numeric_cell">-2.78</td>
                                  <td class="numeric_cell">-1.25</td>
                                  <td class="numeric_cell">0.247</td>
                                  <td class="numeric_cell">0.146</td>
                                  <td class="numeric_cell">0.421</td>
                                  <td class="numeric_cell">2.2e-05</td>
                                  <td class="numeric_cell">0.00207</td>
                                  <td>lm.nb</td>
                                  <td>NM_005194.3:1731</td>
                                </tr>
                                <tr>
                                  <td>VCAM1-mRNA</td>
                                  <td class="numeric_cell">-3.39</td>
                                  <td class="numeric_cell">0.692</td>
                                  <td class="numeric_cell">-4.75</td>
                                  <td class="numeric_cell">-2.04</td>
                                  <td class="numeric_cell">0.0951</td>
                                  <td class="numeric_cell">0.0371</td>
                                  <td class="numeric_cell">0.244</td>
                                  <td class="numeric_cell">4.32e-05</td>
                                  <td class="numeric_cell">0.00366</td>
                                  <td>lm.nb</td>
                                  <td>NM_001078.3:2535</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRB1-mRNA</td>
                                  <td class="numeric_cell">-2.85</td>
                                  <td class="numeric_cell">0.613</td>
                                  <td class="numeric_cell">-4.06</td>
                                  <td class="numeric_cell">-1.65</td>
                                  <td class="numeric_cell">0.138</td>
                                  <td class="numeric_cell">0.0601</td>
                                  <td class="numeric_cell">0.318</td>
                                  <td class="numeric_cell">8.33e-05</td>
                                  <td class="numeric_cell">0.0064</td>
                                  <td>lm.nb</td>
                                  <td>NM_002124.3:1016</td>
                                </tr>
                                <tr>
                                  <td>VEGFA-mRNA</td>
                                  <td class="numeric_cell">-1.61</td>
                                  <td class="numeric_cell">0.353</td>
                                  <td class="numeric_cell">-2.3</td>
                                  <td class="numeric_cell">-0.92</td>
                                  <td class="numeric_cell">0.327</td>
                                  <td class="numeric_cell">0.202</td>
                                  <td class="numeric_cell">0.528</td>
                                  <td class="numeric_cell">0.000105</td>
                                  <td class="numeric_cell">0.00702</td>
                                  <td>lm.nb</td>
                                  <td>NM_001171623.1:2212</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPB1-mRNA</td>
                                  <td class="numeric_cell">-3.33</td>
                                  <td class="numeric_cell">0.731</td>
                                  <td class="numeric_cell">-4.76</td>
                                  <td class="numeric_cell">-1.9</td>
                                  <td class="numeric_cell">0.0993</td>
                                  <td class="numeric_cell">0.0368</td>
                                  <td class="numeric_cell">0.268</td>
                                  <td class="numeric_cell">0.000108</td>
                                  <td class="numeric_cell">0.00702</td>
                                  <td>lm.nb</td>
                                  <td>NM_002121.5:987</td>
                                </tr>
                                <tr>
                                  <td>CD74-mRNA</td>
                                  <td class="numeric_cell">-2.36</td>
                                  <td class="numeric_cell">0.535</td>
                                  <td class="numeric_cell">-3.41</td>
                                  <td class="numeric_cell">-1.31</td>
                                  <td class="numeric_cell">0.195</td>
                                  <td class="numeric_cell">0.0941</td>
                                  <td class="numeric_cell">0.403</td>
                                  <td class="numeric_cell">0.000159</td>
                                  <td class="numeric_cell">0.00959</td>
                                  <td>lm.nb</td>
                                  <td>NM_001025159.2:1161</td>
                                </tr>
                                <tr>
                                  <td>FAM124B-mRNA</td>
                                  <td class="numeric_cell">1.57</td>
                                  <td class="numeric_cell">0.362</td>
                                  <td class="numeric_cell">0.856</td>
                                  <td class="numeric_cell">2.28</td>
                                  <td class="numeric_cell">2.96</td>
                                  <td class="numeric_cell">1.81</td>
                                  <td class="numeric_cell">4.84</td>
                                  <td class="numeric_cell">2e-04</td>
                                  <td class="numeric_cell">0.0113</td>
                                  <td>lm.nb</td>
                                  <td>NM_001122779.1:1620</td>
                                </tr>
                                <tr>
                                  <td>ITGAV-mRNA</td>
                                  <td class="numeric_cell">-2.75</td>
                                  <td class="numeric_cell">0.659</td>
                                  <td class="numeric_cell">-4.04</td>
                                  <td class="numeric_cell">-1.45</td>
                                  <td class="numeric_cell">0.149</td>
                                  <td class="numeric_cell">0.0608</td>
                                  <td class="numeric_cell">0.365</td>
                                  <td class="numeric_cell">0.000303</td>
                                  <td class="numeric_cell">0.0155</td>
                                  <td>lm.nb</td>
                                  <td>NM_001144999.2:2701</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPA1-mRNA</td>
                                  <td class="numeric_cell">-3.64</td>
                                  <td class="numeric_cell">0.879</td>
                                  <td class="numeric_cell">-5.37</td>
                                  <td class="numeric_cell">-1.92</td>
                                  <td class="numeric_cell">0.08</td>
                                  <td class="numeric_cell">0.0242</td>
                                  <td class="numeric_cell">0.264</td>
                                  <td class="numeric_cell">0.000322</td>
                                  <td class="numeric_cell">0.0155</td>
                                  <td>lm.nb</td>
                                  <td>NM_033554.2:857</td>
                                </tr>
                                <tr>
                                  <td>CCL3/L1-mRNA</td>
                                  <td class="numeric_cell">-3.91</td>
                                  <td class="numeric_cell">0.946</td>
                                  <td class="numeric_cell">-5.76</td>
                                  <td class="numeric_cell">-2.05</td>
                                  <td class="numeric_cell">0.0666</td>
                                  <td class="numeric_cell">0.0184</td>
                                  <td class="numeric_cell">0.241</td>
                                  <td class="numeric_cell">0.000331</td>
                                  <td class="numeric_cell">0.0155</td>
                                  <td>lm.nb</td>
                                  <td>NM_021006.5:411</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRA-mRNA</td>
                                  <td class="numeric_cell">-3.1</td>
                                  <td class="numeric_cell">0.779</td>
                                  <td class="numeric_cell">-4.63</td>
                                  <td class="numeric_cell">-1.58</td>
                                  <td class="numeric_cell">0.116</td>
                                  <td class="numeric_cell">0.0404</td>
                                  <td class="numeric_cell">0.335</td>
                                  <td class="numeric_cell">0.000488</td>
                                  <td class="numeric_cell">0.0209</td>
                                  <td>lm.nb</td>
                                  <td>NM_019111.3:335</td>
                                </tr>
                                <tr>
                                  <td>CCL4-mRNA</td>
                                  <td class="numeric_cell">-2.71</td>
                                  <td class="numeric_cell">0.686</td>
                                  <td class="numeric_cell">-4.06</td>
                                  <td class="numeric_cell">-1.37</td>
                                  <td class="numeric_cell">0.153</td>
                                  <td class="numeric_cell">0.0601</td>
                                  <td class="numeric_cell">0.387</td>
                                  <td class="numeric_cell">0.000526</td>
                                  <td class="numeric_cell">0.0209</td>
                                  <td>lm.nb</td>
                                  <td>NM_002984.2:35</td>
                                </tr>
                              </tbody>
                            </table><div id="UK6pdMEHKbJBk9cwqKLAbwg_UKZeCryr1hHv4Wh6W8cAWyA" class="annotation_pop_up"><h5>DE Results: Drug: IFNy vs.Media</h5><p>Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function.</p></div></li>
                            <p>  <br /> <br /></p>
                          
                        </ul></div></div>
                      
                    
                    
                      
                        <div id="UImlUPt-OzhuDCQwu7_f-tg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                          
                            <li><p><b>Volcano Plot: Drug: IgG vs.Media<br /><a href="#" onclick="openPopup('U13nWZ4nPj-rwES2iGGSYNA_UUJgxq3L02shLeynfJQ-7Iw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UUJgxq3L02shLeynfJQ-7Iw');return false;"><div class="small-12 columns"><image src="./results/DE/volcano plotDrugIgG.png" /></div></a><div id="UUJgxq3L02shLeynfJQ-7Iw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/DE/volcano plotDrugIgG.png" style="width: 100%;" /></div></div><div id="U13nWZ4nPj-rwES2iGGSYNA_UUJgxq3L02shLeynfJQ-7Iw" class="annotation_pop_up"><h5>Volcano Plot: Drug: IgG vs.Media</h5><p>Volcano plot displaying each gene's -log10(p-value) and log2 fold change with the selected covariate. Highly statistically significant genes fall at the top of the plot above the horizontal lines, and highly differentially expressed genes fall to either side. Horizontal lines indicate various False Discovery Rate (FDR) thresholds or p-value thresholds if there is no adjustment to the p-values. Genes are colored if the resulting p-value is below the given FDR or p-value threshold. The 40 most statistically significant genes are labeled in the plot.</p></div></li>
                            <li><p><b>DE Results: Drug: IgG vs.Media<br /><a href="#" onclick="openPopup('UK6pdMEHKbJBk9cwqKLAbwg_UqOJgYH9-SrhQMUlqgzxIuQ');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/DE/DE Results - DrugIgG.csv"><span class="label round">Download CSV Data</span></a></b></p><table id="UqOJgYH9-SrhQMUlqgzxIuQ" class="display" cellspacing="0">
                              <Content Count="2">
                                <Annotation Id="UK6pdMEHKbJBk9cwqKLAbwg" Title="Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function." />
                                <FileDownloadLink Id="U8QVTE3iFSZZqlab2ApDwNg" uri="./results/DE/DE Results - DrugIgG.csv" Title="Download CSV Data" />
                              </Content>
                              <thead>
                                <tr>
                                  <td />
                                  <td>Log2 fold change</td>
                                  <td>std error (log2)</td>
                                  <td>Lower confidence limit (log2)</td>
                                  <td>Upper confidence limit (log2)</td>
                                  <td>Linear fold change</td>
                                  <td>Lower confidence limit (linear)</td>
                                  <td>Upper confidence limit (linear)</td>
                                  <td>P-value</td>
                                  <td>BY.p.value</td>
                                  <td>method</td>
                                  <td>probe.ID</td>
                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td>ATF3-mRNA</td>
                                  <td class="numeric_cell">-3.8</td>
                                  <td class="numeric_cell">0.343</td>
                                  <td class="numeric_cell">-4.47</td>
                                  <td class="numeric_cell">-3.13</td>
                                  <td class="numeric_cell">0.0719</td>
                                  <td class="numeric_cell">0.0451</td>
                                  <td class="numeric_cell">0.114</td>
                                  <td class="numeric_cell">2.36e-11</td>
                                  <td class="numeric_cell">1.99e-08</td>
                                  <td>lm.nb</td>
                                  <td>NM_001674.3:1487</td>
                                </tr>
                                <tr>
                                  <td>EGR1-mRNA</td>
                                  <td class="numeric_cell">-3</td>
                                  <td class="numeric_cell">0.394</td>
                                  <td class="numeric_cell">-3.77</td>
                                  <td class="numeric_cell">-2.23</td>
                                  <td class="numeric_cell">0.125</td>
                                  <td class="numeric_cell">0.0732</td>
                                  <td class="numeric_cell">0.213</td>
                                  <td class="numeric_cell">4.32e-08</td>
                                  <td class="numeric_cell">1.34e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_001964.2:2034</td>
                                </tr>
                                <tr>
                                  <td>CD74-mRNA</td>
                                  <td class="numeric_cell">-4.09</td>
                                  <td class="numeric_cell">0.539</td>
                                  <td class="numeric_cell">-5.14</td>
                                  <td class="numeric_cell">-3.03</td>
                                  <td class="numeric_cell">0.0589</td>
                                  <td class="numeric_cell">0.0283</td>
                                  <td class="numeric_cell">0.122</td>
                                  <td class="numeric_cell">4.76e-08</td>
                                  <td class="numeric_cell">1.34e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_001025159.2:1161</td>
                                </tr>
                                <tr>
                                  <td>CXCL14-mRNA</td>
                                  <td class="numeric_cell">-2.76</td>
                                  <td class="numeric_cell">0.373</td>
                                  <td class="numeric_cell">-3.49</td>
                                  <td class="numeric_cell">-2.02</td>
                                  <td class="numeric_cell">0.148</td>
                                  <td class="numeric_cell">0.0892</td>
                                  <td class="numeric_cell">0.246</td>
                                  <td class="numeric_cell">7.62e-08</td>
                                  <td class="numeric_cell">1.61e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_004887.4:990</td>
                                </tr>
                                <tr>
                                  <td>SGK1-mRNA</td>
                                  <td class="numeric_cell">-2.35</td>
                                  <td class="numeric_cell">0.324</td>
                                  <td class="numeric_cell">-2.98</td>
                                  <td class="numeric_cell">-1.71</td>
                                  <td class="numeric_cell">0.196</td>
                                  <td class="numeric_cell">0.126</td>
                                  <td class="numeric_cell">0.305</td>
                                  <td class="numeric_cell">1.06e-07</td>
                                  <td class="numeric_cell">1.8e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_005627.3:1621</td>
                                </tr>
                                <tr>
                                  <td>IRF1-mRNA</td>
                                  <td class="numeric_cell">-2.71</td>
                                  <td class="numeric_cell">0.385</td>
                                  <td class="numeric_cell">-3.46</td>
                                  <td class="numeric_cell">-1.95</td>
                                  <td class="numeric_cell">0.153</td>
                                  <td class="numeric_cell">0.0907</td>
                                  <td class="numeric_cell">0.258</td>
                                  <td class="numeric_cell">1.81e-07</td>
                                  <td class="numeric_cell">2.54e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_002198.2:557</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPB1-mRNA</td>
                                  <td class="numeric_cell">-4.93</td>
                                  <td class="numeric_cell">0.741</td>
                                  <td class="numeric_cell">-6.38</td>
                                  <td class="numeric_cell">-3.47</td>
                                  <td class="numeric_cell">0.0329</td>
                                  <td class="numeric_cell">0.012</td>
                                  <td class="numeric_cell">0.09</td>
                                  <td class="numeric_cell">4.72e-07</td>
                                  <td class="numeric_cell">5.69e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_002121.5:987</td>
                                </tr>
                                <tr>
                                  <td>NFKBIA-mRNA</td>
                                  <td class="numeric_cell">-4.4</td>
                                  <td class="numeric_cell">0.672</td>
                                  <td class="numeric_cell">-5.71</td>
                                  <td class="numeric_cell">-3.08</td>
                                  <td class="numeric_cell">0.0475</td>
                                  <td class="numeric_cell">0.0191</td>
                                  <td class="numeric_cell">0.118</td>
                                  <td class="numeric_cell">6.18e-07</td>
                                  <td class="numeric_cell">6.24e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_020529.2:1361</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPA1-mRNA</td>
                                  <td class="numeric_cell">-5.81</td>
                                  <td class="numeric_cell">0.892</td>
                                  <td class="numeric_cell">-7.56</td>
                                  <td class="numeric_cell">-4.06</td>
                                  <td class="numeric_cell">0.0178</td>
                                  <td class="numeric_cell">0.0053</td>
                                  <td class="numeric_cell">0.0599</td>
                                  <td class="numeric_cell">6.65e-07</td>
                                  <td class="numeric_cell">6.24e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_033554.2:857</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRA-mRNA</td>
                                  <td class="numeric_cell">-4.94</td>
                                  <td class="numeric_cell">0.783</td>
                                  <td class="numeric_cell">-6.47</td>
                                  <td class="numeric_cell">-3.41</td>
                                  <td class="numeric_cell">0.0326</td>
                                  <td class="numeric_cell">0.0113</td>
                                  <td class="numeric_cell">0.0944</td>
                                  <td class="numeric_cell">1.11e-06</td>
                                  <td class="numeric_cell">9.4e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_019111.3:335</td>
                                </tr>
                                <tr>
                                  <td>DUSP1-mRNA</td>
                                  <td class="numeric_cell">-4.37</td>
                                  <td class="numeric_cell">0.753</td>
                                  <td class="numeric_cell">-5.85</td>
                                  <td class="numeric_cell">-2.9</td>
                                  <td class="numeric_cell">0.0482</td>
                                  <td class="numeric_cell">0.0173</td>
                                  <td class="numeric_cell">0.134</td>
                                  <td class="numeric_cell">4.06e-06</td>
                                  <td class="numeric_cell">0.000312</td>
                                  <td>lm.nb</td>
                                  <td>NM_004417.3:1428</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRB1-mRNA</td>
                                  <td class="numeric_cell">-2.98</td>
                                  <td class="numeric_cell">0.614</td>
                                  <td class="numeric_cell">-4.18</td>
                                  <td class="numeric_cell">-1.77</td>
                                  <td class="numeric_cell">0.127</td>
                                  <td class="numeric_cell">0.0552</td>
                                  <td class="numeric_cell">0.293</td>
                                  <td class="numeric_cell">5.01e-05</td>
                                  <td class="numeric_cell">0.00353</td>
                                  <td>lm.nb</td>
                                  <td>NM_002124.3:1016</td>
                                </tr>
                                <tr>
                                  <td>TGFB1-mRNA</td>
                                  <td class="numeric_cell">-1.9</td>
                                  <td class="numeric_cell">0.399</td>
                                  <td class="numeric_cell">-2.68</td>
                                  <td class="numeric_cell">-1.12</td>
                                  <td class="numeric_cell">0.268</td>
                                  <td class="numeric_cell">0.156</td>
                                  <td class="numeric_cell">0.461</td>
                                  <td class="numeric_cell">6.31e-05</td>
                                  <td class="numeric_cell">0.0041</td>
                                  <td>lm.nb</td>
                                  <td>NM_000660.4:1473</td>
                                </tr>
                                <tr>
                                  <td>CEBPB-mRNA</td>
                                  <td class="numeric_cell">-1.78</td>
                                  <td class="numeric_cell">0.388</td>
                                  <td class="numeric_cell">-2.54</td>
                                  <td class="numeric_cell">-1.02</td>
                                  <td class="numeric_cell">0.29</td>
                                  <td class="numeric_cell">0.171</td>
                                  <td class="numeric_cell">0.492</td>
                                  <td class="numeric_cell">9.64e-05</td>
                                  <td class="numeric_cell">0.00582</td>
                                  <td>lm.nb</td>
                                  <td>NM_005194.3:1731</td>
                                </tr>
                                <tr>
                                  <td>GBP2-mRNA</td>
                                  <td class="numeric_cell">-2.6</td>
                                  <td class="numeric_cell">0.577</td>
                                  <td class="numeric_cell">-3.73</td>
                                  <td class="numeric_cell">-1.47</td>
                                  <td class="numeric_cell">0.165</td>
                                  <td class="numeric_cell">0.0754</td>
                                  <td class="numeric_cell">0.362</td>
                                  <td class="numeric_cell">0.000126</td>
                                  <td class="numeric_cell">0.00702</td>
                                  <td>lm.nb</td>
                                  <td>NM_004120.4:1744</td>
                                </tr>
                                <tr>
                                  <td>HLA-E-mRNA</td>
                                  <td class="numeric_cell">-3.07</td>
                                  <td class="numeric_cell">0.687</td>
                                  <td class="numeric_cell">-4.42</td>
                                  <td class="numeric_cell">-1.72</td>
                                  <td class="numeric_cell">0.119</td>
                                  <td class="numeric_cell">0.0468</td>
                                  <td class="numeric_cell">0.303</td>
                                  <td class="numeric_cell">0.000136</td>
                                  <td class="numeric_cell">0.00702</td>
                                  <td>lm.nb</td>
                                  <td>NM_005516.5:1287</td>
                                </tr>
                                <tr>
                                  <td>CDKN1A-mRNA</td>
                                  <td class="numeric_cell">-1.26</td>
                                  <td class="numeric_cell">0.283</td>
                                  <td class="numeric_cell">-1.82</td>
                                  <td class="numeric_cell">-0.707</td>
                                  <td class="numeric_cell">0.417</td>
                                  <td class="numeric_cell">0.284</td>
                                  <td class="numeric_cell">0.613</td>
                                  <td class="numeric_cell">0.000141</td>
                                  <td class="numeric_cell">0.00702</td>
                                  <td>lm.nb</td>
                                  <td>NM_000389.2:1975</td>
                                </tr>
                                <tr>
                                  <td>THBS1-mRNA</td>
                                  <td class="numeric_cell">-3.37</td>
                                  <td class="numeric_cell">0.765</td>
                                  <td class="numeric_cell">-4.87</td>
                                  <td class="numeric_cell">-1.87</td>
                                  <td class="numeric_cell">0.0967</td>
                                  <td class="numeric_cell">0.0342</td>
                                  <td class="numeric_cell">0.273</td>
                                  <td class="numeric_cell">0.000161</td>
                                  <td class="numeric_cell">0.00732</td>
                                  <td>lm.nb</td>
                                  <td>NM_003246.2:3816</td>
                                </tr>
                                <tr>
                                  <td>STAT1-mRNA</td>
                                  <td class="numeric_cell">-1.44</td>
                                  <td class="numeric_cell">0.326</td>
                                  <td class="numeric_cell">-2.08</td>
                                  <td class="numeric_cell">-0.796</td>
                                  <td class="numeric_cell">0.37</td>
                                  <td class="numeric_cell">0.237</td>
                                  <td class="numeric_cell">0.576</td>
                                  <td class="numeric_cell">0.000164</td>
                                  <td class="numeric_cell">0.00732</td>
                                  <td>lm.nb</td>
                                  <td>NM_007315.3:239</td>
                                </tr>
                                <tr>
                                  <td>APOE-mRNA</td>
                                  <td class="numeric_cell">-2.91</td>
                                  <td class="numeric_cell">0.669</td>
                                  <td class="numeric_cell">-4.22</td>
                                  <td class="numeric_cell">-1.6</td>
                                  <td class="numeric_cell">0.133</td>
                                  <td class="numeric_cell">0.0537</td>
                                  <td class="numeric_cell">0.331</td>
                                  <td class="numeric_cell">0.00019</td>
                                  <td class="numeric_cell">0.00804</td>
                                  <td>lm.nb</td>
                                  <td>NM_000041.2:96</td>
                                </tr>
                              </tbody>
                            </table><div id="UK6pdMEHKbJBk9cwqKLAbwg_UqOJgYH9-SrhQMUlqgzxIuQ" class="annotation_pop_up"><h5>DE Results: Drug: IgG vs.Media</h5><p>Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function.</p></div></li>
                            <p>  <br /> <br /></p>
                          
                        </ul></div></div>
                      
                    
                    
                      
                        <div id="UP734xI-HVQYhnax6P4UkJw" class="content"><div class="small-12 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                          
                            <li><p><b>Volcano Plot: Drug: NMC2 vs.Media<br /><a href="#" onclick="openPopup('U13nWZ4nPj-rwES2iGGSYNA_UrfQSFTaXNx-fuZF0vBWzqw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UrfQSFTaXNx-fuZF0vBWzqw');return false;"><div class="small-12 columns"><image src="./results/DE/volcano plotDrugNMC2.png" /></div></a><div id="UrfQSFTaXNx-fuZF0vBWzqw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/DE/volcano plotDrugNMC2.png" style="width: 100%;" /></div></div><div id="U13nWZ4nPj-rwES2iGGSYNA_UrfQSFTaXNx-fuZF0vBWzqw" class="annotation_pop_up"><h5>Volcano Plot: Drug: NMC2 vs.Media</h5><p>Volcano plot displaying each gene's -log10(p-value) and log2 fold change with the selected covariate. Highly statistically significant genes fall at the top of the plot above the horizontal lines, and highly differentially expressed genes fall to either side. Horizontal lines indicate various False Discovery Rate (FDR) thresholds or p-value thresholds if there is no adjustment to the p-values. Genes are colored if the resulting p-value is below the given FDR or p-value threshold. The 40 most statistically significant genes are labeled in the plot.</p></div></li>
                            <li><p><b>DE Results: Drug: NMC2 vs.Media<br /><a href="#" onclick="openPopup('UK6pdMEHKbJBk9cwqKLAbwg_U8Y0sc_EDVSi0x19uW201Rw');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/DE/DE Results - DrugNMC2.csv"><span class="label round">Download CSV Data</span></a></b></p><table id="U8Y0sc_EDVSi0x19uW201Rw" class="display" cellspacing="0">
                              <Content Count="2">
                                <Annotation Id="UK6pdMEHKbJBk9cwqKLAbwg" Title="Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function." />
                                <FileDownloadLink Id="Uw4qVAZE3ePO9XOVXfvp4Pw" uri="./results/DE/DE Results - DrugNMC2.csv" Title="Download CSV Data" />
                              </Content>
                              <thead>
                                <tr>
                                  <td />
                                  <td>Log2 fold change</td>
                                  <td>std error (log2)</td>
                                  <td>Lower confidence limit (log2)</td>
                                  <td>Upper confidence limit (log2)</td>
                                  <td>Linear fold change</td>
                                  <td>Lower confidence limit (linear)</td>
                                  <td>Upper confidence limit (linear)</td>
                                  <td>P-value</td>
                                  <td>BY.p.value</td>
                                  <td>method</td>
                                  <td>probe.ID</td>
                                </tr>
                              </thead>
                              <tbody>
                                <tr>
                                  <td>ATF3-mRNA</td>
                                  <td class="numeric_cell">-3.17</td>
                                  <td class="numeric_cell">0.283</td>
                                  <td class="numeric_cell">-3.73</td>
                                  <td class="numeric_cell">-2.62</td>
                                  <td class="numeric_cell">0.111</td>
                                  <td class="numeric_cell">0.0755</td>
                                  <td class="numeric_cell">0.163</td>
                                  <td class="numeric_cell">1.83e-11</td>
                                  <td class="numeric_cell">1.55e-08</td>
                                  <td>lm.nb</td>
                                  <td>NM_001674.3:1487</td>
                                </tr>
                                <tr>
                                  <td>CD74-mRNA</td>
                                  <td class="numeric_cell">-4.08</td>
                                  <td class="numeric_cell">0.465</td>
                                  <td class="numeric_cell">-4.99</td>
                                  <td class="numeric_cell">-3.17</td>
                                  <td class="numeric_cell">0.0591</td>
                                  <td class="numeric_cell">0.0314</td>
                                  <td class="numeric_cell">0.111</td>
                                  <td class="numeric_cell">3.02e-09</td>
                                  <td class="numeric_cell">1.18e-06</td>
                                  <td>lm.nb</td>
                                  <td>NM_001025159.2:1161</td>
                                </tr>
                                <tr>
                                  <td>IRF1-mRNA</td>
                                  <td class="numeric_cell">-2.82</td>
                                  <td class="numeric_cell">0.327</td>
                                  <td class="numeric_cell">-3.46</td>
                                  <td class="numeric_cell">-2.18</td>
                                  <td class="numeric_cell">0.141</td>
                                  <td class="numeric_cell">0.0907</td>
                                  <td class="numeric_cell">0.221</td>
                                  <td class="numeric_cell">4.2e-09</td>
                                  <td class="numeric_cell">1.18e-06</td>
                                  <td>lm.nb</td>
                                  <td>NM_002198.2:557</td>
                                </tr>
                                <tr>
                                  <td>CXCL14-mRNA</td>
                                  <td class="numeric_cell">-2.55</td>
                                  <td class="numeric_cell">0.314</td>
                                  <td class="numeric_cell">-3.16</td>
                                  <td class="numeric_cell">-1.93</td>
                                  <td class="numeric_cell">0.171</td>
                                  <td class="numeric_cell">0.112</td>
                                  <td class="numeric_cell">0.262</td>
                                  <td class="numeric_cell">1.38e-08</td>
                                  <td class="numeric_cell">2.93e-06</td>
                                  <td>lm.nb</td>
                                  <td>NM_004887.4:990</td>
                                </tr>
                                <tr>
                                  <td>SGK1-mRNA</td>
                                  <td class="numeric_cell">-2.18</td>
                                  <td class="numeric_cell">0.274</td>
                                  <td class="numeric_cell">-2.72</td>
                                  <td class="numeric_cell">-1.65</td>
                                  <td class="numeric_cell">0.22</td>
                                  <td class="numeric_cell">0.152</td>
                                  <td class="numeric_cell">0.32</td>
                                  <td class="numeric_cell">1.94e-08</td>
                                  <td class="numeric_cell">3.29e-06</td>
                                  <td>lm.nb</td>
                                  <td>NM_005627.3:1621</td>
                                </tr>
                                <tr>
                                  <td>EGR1-mRNA</td>
                                  <td class="numeric_cell">-2.53</td>
                                  <td class="numeric_cell">0.335</td>
                                  <td class="numeric_cell">-3.19</td>
                                  <td class="numeric_cell">-1.87</td>
                                  <td class="numeric_cell">0.173</td>
                                  <td class="numeric_cell">0.11</td>
                                  <td class="numeric_cell">0.273</td>
                                  <td class="numeric_cell">5.19e-08</td>
                                  <td class="numeric_cell">7.31e-06</td>
                                  <td>lm.nb</td>
                                  <td>NM_001964.2:2034</td>
                                </tr>
                                <tr>
                                  <td>NFKBIA-mRNA</td>
                                  <td class="numeric_cell">-4.12</td>
                                  <td class="numeric_cell">0.578</td>
                                  <td class="numeric_cell">-5.25</td>
                                  <td class="numeric_cell">-2.99</td>
                                  <td class="numeric_cell">0.0574</td>
                                  <td class="numeric_cell">0.0262</td>
                                  <td class="numeric_cell">0.126</td>
                                  <td class="numeric_cell">1.4e-07</td>
                                  <td class="numeric_cell">1.61e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_020529.2:1361</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPB1-mRNA</td>
                                  <td class="numeric_cell">-4.51</td>
                                  <td class="numeric_cell">0.635</td>
                                  <td class="numeric_cell">-5.76</td>
                                  <td class="numeric_cell">-3.27</td>
                                  <td class="numeric_cell">0.0438</td>
                                  <td class="numeric_cell">0.0185</td>
                                  <td class="numeric_cell">0.104</td>
                                  <td class="numeric_cell">1.52e-07</td>
                                  <td class="numeric_cell">1.61e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_002121.5:987</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRA-mRNA</td>
                                  <td class="numeric_cell">-4.73</td>
                                  <td class="numeric_cell">0.676</td>
                                  <td class="numeric_cell">-6.06</td>
                                  <td class="numeric_cell">-3.41</td>
                                  <td class="numeric_cell">0.0377</td>
                                  <td class="numeric_cell">0.015</td>
                                  <td class="numeric_cell">0.0944</td>
                                  <td class="numeric_cell">1.98e-07</td>
                                  <td class="numeric_cell">1.86e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_019111.3:335</td>
                                </tr>
                                <tr>
                                  <td>DUSP1-mRNA</td>
                                  <td class="numeric_cell">-4.46</td>
                                  <td class="numeric_cell">0.651</td>
                                  <td class="numeric_cell">-5.74</td>
                                  <td class="numeric_cell">-3.18</td>
                                  <td class="numeric_cell">0.0454</td>
                                  <td class="numeric_cell">0.0187</td>
                                  <td class="numeric_cell">0.11</td>
                                  <td class="numeric_cell">2.86e-07</td>
                                  <td class="numeric_cell">2.42e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_004417.3:1428</td>
                                </tr>
                                <tr>
                                  <td>HLA-DPA1-mRNA</td>
                                  <td class="numeric_cell">-4.85</td>
                                  <td class="numeric_cell">0.763</td>
                                  <td class="numeric_cell">-6.35</td>
                                  <td class="numeric_cell">-3.36</td>
                                  <td class="numeric_cell">0.0346</td>
                                  <td class="numeric_cell">0.0123</td>
                                  <td class="numeric_cell">0.0977</td>
                                  <td class="numeric_cell">9.89e-07</td>
                                  <td class="numeric_cell">7.6e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_033554.2:857</td>
                                </tr>
                                <tr>
                                  <td>CEBPB-mRNA</td>
                                  <td class="numeric_cell">-2.1</td>
                                  <td class="numeric_cell">0.335</td>
                                  <td class="numeric_cell">-2.76</td>
                                  <td class="numeric_cell">-1.45</td>
                                  <td class="numeric_cell">0.233</td>
                                  <td class="numeric_cell">0.148</td>
                                  <td class="numeric_cell">0.367</td>
                                  <td class="numeric_cell">1.2e-06</td>
                                  <td class="numeric_cell">8.45e-05</td>
                                  <td>lm.nb</td>
                                  <td>NM_005194.3:1731</td>
                                </tr>
                                <tr>
                                  <td>HLA-DRB1-mRNA</td>
                                  <td class="numeric_cell">-3.13</td>
                                  <td class="numeric_cell">0.53</td>
                                  <td class="numeric_cell">-4.17</td>
                                  <td class="numeric_cell">-2.1</td>
                                  <td class="numeric_cell">0.114</td>
                                  <td class="numeric_cell">0.0554</td>
                                  <td class="numeric_cell">0.234</td>
                                  <td class="numeric_cell">3.09e-06</td>
                                  <td class="numeric_cell">0.000201</td>
                                  <td>lm.nb</td>
                                  <td>NM_002124.3:1016</td>
                                </tr>
                                <tr>
                                  <td>VCAM1-mRNA</td>
                                  <td class="numeric_cell">-3.32</td>
                                  <td class="numeric_cell">0.595</td>
                                  <td class="numeric_cell">-4.49</td>
                                  <td class="numeric_cell">-2.15</td>
                                  <td class="numeric_cell">0.1</td>
                                  <td class="numeric_cell">0.0446</td>
                                  <td class="numeric_cell">0.225</td>
                                  <td class="numeric_cell">7.43e-06</td>
                                  <td class="numeric_cell">0.000448</td>
                                  <td>lm.nb</td>
                                  <td>NM_001078.3:2535</td>
                                </tr>
                                <tr>
                                  <td>GBP2-mRNA</td>
                                  <td class="numeric_cell">-2.74</td>
                                  <td class="numeric_cell">0.496</td>
                                  <td class="numeric_cell">-3.71</td>
                                  <td class="numeric_cell">-1.77</td>
                                  <td class="numeric_cell">0.15</td>
                                  <td class="numeric_cell">0.0763</td>
                                  <td class="numeric_cell">0.293</td>
                                  <td class="numeric_cell">8.3e-06</td>
                                  <td class="numeric_cell">0.000468</td>
                                  <td>lm.nb</td>
                                  <td>NM_004120.4:1744</td>
                                </tr>
                                <tr>
                                  <td>TGFB1-mRNA</td>
                                  <td class="numeric_cell">-1.72</td>
                                  <td class="numeric_cell">0.34</td>
                                  <td class="numeric_cell">-2.39</td>
                                  <td class="numeric_cell">-1.06</td>
                                  <td class="numeric_cell">0.303</td>
                                  <td class="numeric_cell">0.191</td>
                                  <td class="numeric_cell">0.48</td>
                                  <td class="numeric_cell">2.81e-05</td>
                                  <td class="numeric_cell">0.00148</td>
                                  <td>lm.nb</td>
                                  <td>NM_000660.4:1473</td>
                                </tr>
                                <tr>
                                  <td>VEGFA-mRNA</td>
                                  <td class="numeric_cell">-1.51</td>
                                  <td class="numeric_cell">0.301</td>
                                  <td class="numeric_cell">-2.1</td>
                                  <td class="numeric_cell">-0.923</td>
                                  <td class="numeric_cell">0.35</td>
                                  <td class="numeric_cell">0.233</td>
                                  <td class="numeric_cell">0.527</td>
                                  <td class="numeric_cell">3.11e-05</td>
                                  <td class="numeric_cell">0.00155</td>
                                  <td>lm.nb</td>
                                  <td>NM_001171623.1:2212</td>
                                </tr>
                                <tr>
                                  <td>STAT1-mRNA</td>
                                  <td class="numeric_cell">-1.38</td>
                                  <td class="numeric_cell">0.279</td>
                                  <td class="numeric_cell">-1.93</td>
                                  <td class="numeric_cell">-0.836</td>
                                  <td class="numeric_cell">0.383</td>
                                  <td class="numeric_cell">0.262</td>
                                  <td class="numeric_cell">0.56</td>
                                  <td class="numeric_cell">3.79e-05</td>
                                  <td class="numeric_cell">0.00178</td>
                                  <td>lm.nb</td>
                                  <td>NM_007315.3:239</td>
                                </tr>
                                <tr>
                                  <td>MS4A1-mRNA</td>
                                  <td class="numeric_cell">1.79</td>
                                  <td class="numeric_cell">0.363</td>
                                  <td class="numeric_cell">1.08</td>
                                  <td class="numeric_cell">2.51</td>
                                  <td class="numeric_cell">3.47</td>
                                  <td class="numeric_cell">2.12</td>
                                  <td class="numeric_cell">5.68</td>
                                  <td class="numeric_cell">3.99e-05</td>
                                  <td class="numeric_cell">0.00178</td>
                                  <td>lm.nb</td>
                                  <td>NM_021950.3:161</td>
                                </tr>
                                <tr>
                                  <td>THBS1-mRNA</td>
                                  <td class="numeric_cell">-3.17</td>
                                  <td class="numeric_cell">0.659</td>
                                  <td class="numeric_cell">-4.46</td>
                                  <td class="numeric_cell">-1.88</td>
                                  <td class="numeric_cell">0.111</td>
                                  <td class="numeric_cell">0.0455</td>
                                  <td class="numeric_cell">0.273</td>
                                  <td class="numeric_cell">5.6e-05</td>
                                  <td class="numeric_cell">0.00237</td>
                                  <td>lm.nb</td>
                                  <td>NM_003246.2:3816</td>
                                </tr>
                              </tbody>
                            </table><div id="UK6pdMEHKbJBk9cwqKLAbwg_U8Y0sc_EDVSi0x19uW201Rw" class="annotation_pop_up"><h5>DE Results: Drug: NMC2 vs.Media</h5><p>Table presenting the most statistically significantly differentially expressed genes with the selected covariate. 'Estimated log fold-change' estimates a gene's differential expression. For categorical covariates, a gene is estimated to have 2^(log fold change) times its expression in baseline samples, holding all other variables in the analysis constant. If the covariate is continuous, for each unit increase in the selected covariate, a gene's expression is estimated to increase by 2^(log fold change)-fold, holding all other variables in the analysis constant. The 95% confidence interval for the log fold change is also presented, along with a p-value and an adjusted p-value or FDR if requested. Method column indicates the model used to estimate differential expression. Mixed negative binomial model uses the mle function to run the Wald test. The simplified negative binomial model uses the glm.nb function . The loglinear model uses the lm function.</p></div></li>
                            <p>  <br /> <br /></p>
                          
                        </ul></div></div>
                      
                    
                  
                </div>
              
            </ul></div></div></div>
          
        
        
          
            <div id="U9YhWvqi0JdlimyJ-bFsxwA" class="panelContent" style="display:none"><div class="row"><div class="panel"><p>Pathview is used to overlay differential expression results on KEGG pathways</p></div></div><div class="row"><div class="panel"><ul class="small-block-grid-1">
              
                <dl class="sub-nav" data-tab=""><dt>Drug:</dt>
                  
                    <li class="active"><a menuindex="Drug|Drug: differential expression in IFNy vs. baseline of Media" href="#UpmksSq99pYTZe3mkR7H8Bg">Drug: differential expression in IFNy vs. baseline of Media</a></li>
                    <li><a menuindex="Drug|Drug: differential expression in IgG vs. baseline of Media" href="#UVwD34DrhlJe4FLRRfyfWSg">Drug: differential expression in IgG vs. baseline of Media</a></li>
                    <li><a menuindex="Drug|Drug: differential expression in NMC2 vs. baseline of Media" href="#U_MQV51xZuNaIOCbS3bAznQ">Drug: differential expression in NMC2 vs. baseline of Media</a></li>
                  
                </dl><div class="tabs-content">
                  
                    
                      
                        <div id="UpmksSq99pYTZe3mkR7H8Bg" class="content active"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Pathways:</p>
                              
                                <li class="tab-title active"><a href="#U4TjF5PAKBgd97bNBTbTRYw">HTLV-I infection</a></li>
                                <li class="tab-title"><a href="#UxLAa7v0_OCRcrEIrwLn0Rg">Cytokine-cytokine receptor interaction</a></li>
                                <li class="tab-title"><a href="#UJUiihVvWukzx12JFwBrB9w">Tuberculosis</a></li>
                                <li class="tab-title"><a href="#U4YwgKpjYiJj3MIZRWNxw-g">Pathways in cancer</a></li>
                                <li class="tab-title"><a href="#UN90qUkxYIBgfSaZAmhqE0Q">Herpes simplex infection</a></li>
                                <li class="tab-title"><a href="#U6Axbgx9uKTYd0Z5MwUCaNA">Chemokine signaling pathway</a></li>
                                <li class="tab-title"><a href="#UWP7PN0xcLdkZb3lY8rWWgg">Influenza A</a></li>
                                <li class="tab-title"><a href="#UdvoyTIa4yuomrpv6QW_OGQ">Toxoplasmosis</a></li>
                                <li class="tab-title"><a href="#U3k_X4ctEdeThXa2LG8fzOQ">PI3K-Akt signaling pathway</a></li>
                                <li class="tab-title"><a href="#UpgCByUozcVCaFLyTVr7bXA">Cell adhesion molecules (CAMs)</a></li>
                                <li class="tab-title"><a href="#UiyL9wLZnM-H_aLktvpNpYA">Osteoclast differentiation</a></li>
                                <li class="tab-title"><a href="#U3-G6SSYoGKdfBJpSWCw2Pg">Epstein-Barr virus infection</a></li>
                                <li class="tab-title"><a href="#UlCt0SDXIxCC5SGb0trfoZQ">Proteoglycans in cancer</a></li>
                                <li class="tab-title"><a href="#UYYiP2QIRgS599GCR85BkNQ">Natural killer cell mediated cytotoxicity</a></li>
                                <li class="tab-title"><a href="#USuphhX1DM-WTrZCWUmQADw">TNF signaling pathway</a></li>
                                <li class="tab-title"><a href="#Ues6cGgTcKBc2VBlxYA6QbQ">Measles</a></li>
                                <li class="tab-title"><a href="#UIbf6OXk6PSrRen8RFTU69w">Hepatitis B</a></li>
                                <li class="tab-title"><a href="#UL2duM65wdrpUcqky251ssQ">Viral carcinogenesis</a></li>
                                <li class="tab-title"><a href="#UyWLgO56oDRQukNP2eObgng">Jak-STAT signaling pathway</a></li>
                                <li class="tab-title"><a href="#UbsTxXYpgHsNn9W4TVzu1DQ">MicroRNAs in cancer</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="U4TjF5PAKBgd97bNBTbTRYw" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UkP8DsoW2hTkvQAFRYEUcgA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UkP8DsoW2hTkvQAFRYEUcgA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05166.DrugIFNy.png" /></div></a><div id="UkP8DsoW2hTkvQAFRYEUcgA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05166.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UkP8DsoW2hTkvQAFRYEUcgA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UxLAa7v0_OCRcrEIrwLn0Rg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UWiuPm4K2wbszNav6gaNrew');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UWiuPm4K2wbszNav6gaNrew');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04060.DrugIFNy.png" /></div></a><div id="UWiuPm4K2wbszNav6gaNrew" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04060.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UWiuPm4K2wbszNav6gaNrew" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UJUiihVvWukzx12JFwBrB9w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U1mhC1TtdlE9Bs9fKmlBphQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U1mhC1TtdlE9Bs9fKmlBphQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05152.DrugIFNy.png" /></div></a><div id="U1mhC1TtdlE9Bs9fKmlBphQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05152.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U1mhC1TtdlE9Bs9fKmlBphQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U4YwgKpjYiJj3MIZRWNxw-g" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Uc4zv3a-e3ZsjvNiaGfdiSg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uc4zv3a-e3ZsjvNiaGfdiSg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05200.DrugIFNy.png" /></div></a><div id="Uc4zv3a-e3ZsjvNiaGfdiSg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05200.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Uc4zv3a-e3ZsjvNiaGfdiSg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UN90qUkxYIBgfSaZAmhqE0Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UIY4GgVh3hqnorv2aIeJU1w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UIY4GgVh3hqnorv2aIeJU1w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05168.DrugIFNy.png" /></div></a><div id="UIY4GgVh3hqnorv2aIeJU1w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05168.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UIY4GgVh3hqnorv2aIeJU1w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U6Axbgx9uKTYd0Z5MwUCaNA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UW0AFt2zfBTopvtSnNwe-Jg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UW0AFt2zfBTopvtSnNwe-Jg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04062.DrugIFNy.png" /></div></a><div id="UW0AFt2zfBTopvtSnNwe-Jg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04062.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UW0AFt2zfBTopvtSnNwe-Jg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UWP7PN0xcLdkZb3lY8rWWgg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_ULUX0ruOdqCzeSlnfrgsdVg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('ULUX0ruOdqCzeSlnfrgsdVg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05164.DrugIFNy.png" /></div></a><div id="ULUX0ruOdqCzeSlnfrgsdVg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05164.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_ULUX0ruOdqCzeSlnfrgsdVg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UdvoyTIa4yuomrpv6QW_OGQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Uf-tLOQcP3sma6C1ESBSk7A');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uf-tLOQcP3sma6C1ESBSk7A');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05145.DrugIFNy.png" /></div></a><div id="Uf-tLOQcP3sma6C1ESBSk7A" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05145.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Uf-tLOQcP3sma6C1ESBSk7A" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U3k_X4ctEdeThXa2LG8fzOQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UjyUdRPPicmfhpiStkGy3gA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UjyUdRPPicmfhpiStkGy3gA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04151.DrugIFNy.png" /></div></a><div id="UjyUdRPPicmfhpiStkGy3gA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04151.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UjyUdRPPicmfhpiStkGy3gA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UpgCByUozcVCaFLyTVr7bXA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UPOohnX-MmmFo5BYi8ENc3w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UPOohnX-MmmFo5BYi8ENc3w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04514.DrugIFNy.png" /></div></a><div id="UPOohnX-MmmFo5BYi8ENc3w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04514.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UPOohnX-MmmFo5BYi8ENc3w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UiyL9wLZnM-H_aLktvpNpYA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UyQNp15pmdj19tjIcGwHOhA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UyQNp15pmdj19tjIcGwHOhA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04380.DrugIFNy.png" /></div></a><div id="UyQNp15pmdj19tjIcGwHOhA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04380.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UyQNp15pmdj19tjIcGwHOhA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U3-G6SSYoGKdfBJpSWCw2Pg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UoWgG7PlG0Vei-yrqH9U9OQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UoWgG7PlG0Vei-yrqH9U9OQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05169.DrugIFNy.png" /></div></a><div id="UoWgG7PlG0Vei-yrqH9U9OQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05169.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UoWgG7PlG0Vei-yrqH9U9OQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UlCt0SDXIxCC5SGb0trfoZQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UfcPcapBpphz8SImfHFDGGw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UfcPcapBpphz8SImfHFDGGw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05205.DrugIFNy.png" /></div></a><div id="UfcPcapBpphz8SImfHFDGGw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05205.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UfcPcapBpphz8SImfHFDGGw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UYYiP2QIRgS599GCR85BkNQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UUlHOI3vytqS0-1teUCoRtw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UUlHOI3vytqS0-1teUCoRtw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04650.DrugIFNy.png" /></div></a><div id="UUlHOI3vytqS0-1teUCoRtw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04650.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UUlHOI3vytqS0-1teUCoRtw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="USuphhX1DM-WTrZCWUmQADw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UXFWi0hdJg5YFi5TPA87vvw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UXFWi0hdJg5YFi5TPA87vvw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04668.DrugIFNy.png" /></div></a><div id="UXFWi0hdJg5YFi5TPA87vvw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04668.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UXFWi0hdJg5YFi5TPA87vvw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Ues6cGgTcKBc2VBlxYA6QbQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U9P1m4GTGA4oAqbmGn69-iw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U9P1m4GTGA4oAqbmGn69-iw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05162.DrugIFNy.png" /></div></a><div id="U9P1m4GTGA4oAqbmGn69-iw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05162.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U9P1m4GTGA4oAqbmGn69-iw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UIbf6OXk6PSrRen8RFTU69w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UXXnNwOmrXNaMyIcHcunI5w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UXXnNwOmrXNaMyIcHcunI5w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05161.DrugIFNy.png" /></div></a><div id="UXXnNwOmrXNaMyIcHcunI5w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05161.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UXXnNwOmrXNaMyIcHcunI5w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UL2duM65wdrpUcqky251ssQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UrnJJ_OE-jwYXcJK5JfN-_A');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UrnJJ_OE-jwYXcJK5JfN-_A');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05203.DrugIFNy.png" /></div></a><div id="UrnJJ_OE-jwYXcJK5JfN-_A" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05203.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UrnJJ_OE-jwYXcJK5JfN-_A" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UyWLgO56oDRQukNP2eObgng" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UOt82zraKywYJbMDQHXvKmw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UOt82zraKywYJbMDQHXvKmw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04630.DrugIFNy.png" /></div></a><div id="UOt82zraKywYJbMDQHXvKmw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04630.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UOt82zraKywYJbMDQHXvKmw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UbsTxXYpgHsNn9W4TVzu1DQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UVXiPCVKa2Aj5Oaa74km0Sw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UVXiPCVKa2Aj5Oaa74km0Sw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05206.DrugIFNy.png" /></div></a><div id="UVXiPCVKa2Aj5Oaa74km0Sw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05206.DrugIFNy.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UVXiPCVKa2Aj5Oaa74km0Sw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                    
                      
                        <div id="UVwD34DrhlJe4FLRRfyfWSg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Pathways:</p>
                              
                                <li class="tab-title active"><a href="#UcQiLnPwgBbrQa3LqLTDfTA">HTLV-I infection</a></li>
                                <li class="tab-title"><a href="#UG2f_AnAz5ktiq6SRiFitKw">Herpes simplex infection</a></li>
                                <li class="tab-title"><a href="#UU8LsU_sT2S_lkotEcVc5YQ">Influenza A</a></li>
                                <li class="tab-title"><a href="#UwkbYvqDXUDIOQPaCj2Vy6A">Tuberculosis</a></li>
                                <li class="tab-title"><a href="#Ut6880H6w9QjqBY5YlFvYlg">Cell adhesion molecules (CAMs)</a></li>
                                <li class="tab-title"><a href="#UA75Rn4cQHe7JSLWT1p7K4w">Epstein-Barr virus infection</a></li>
                                <li class="tab-title"><a href="#UkNu0ElVacSX6id5LkWlWjg">Cytokine-cytokine receptor interaction</a></li>
                                <li class="tab-title"><a href="#UNUtdbCj1pXOLr5JZbqbRag">Natural killer cell mediated cytotoxicity</a></li>
                                <li class="tab-title"><a href="#UM2Ev1fwZkPphOablYV_3vw">Pathways in cancer</a></li>
                                <li class="tab-title"><a href="#UVzWewBbHBYYyuH8XiTtGog">Chemokine signaling pathway</a></li>
                                <li class="tab-title"><a href="#UqDSWcV2ZD1S5fF9Vw9fo3w">Toxoplasmosis</a></li>
                                <li class="tab-title"><a href="#UF-uW_de47GqGx2RhgC4DcA">Osteoclast differentiation</a></li>
                                <li class="tab-title"><a href="#UVz6R0_BDm0dYjReyr29GFg">TNF signaling pathway</a></li>
                                <li class="tab-title"><a href="#UEGKp0meZvEikoPhotx9Qmg">Measles</a></li>
                                <li class="tab-title"><a href="#UO0dy4aEkVPkL7YeoeeNNtA">Hepatitis B</a></li>
                                <li class="tab-title"><a href="#UqZcrH56Rwc7fq37dI9jcig">Viral carcinogenesis</a></li>
                                <li class="tab-title"><a href="#U_N7TGEfmlfEdTIxR-VD1YA">Jak-STAT signaling pathway</a></li>
                                <li class="tab-title"><a href="#UsUqf8FJyHI-YevXYhifZig">PI3K-Akt signaling pathway</a></li>
                                <li class="tab-title"><a href="#UAxd2WbGMiKLYv508OZhF9Q">Proteoglycans in cancer</a></li>
                                <li class="tab-title"><a href="#UL4JhuDLHdslx9bcp6Uai2g">MicroRNAs in cancer</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="UcQiLnPwgBbrQa3LqLTDfTA" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Urv-Yz-B1YVa8I0HVpvnGpQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Urv-Yz-B1YVa8I0HVpvnGpQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05166.DrugIgG.png" /></div></a><div id="Urv-Yz-B1YVa8I0HVpvnGpQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05166.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Urv-Yz-B1YVa8I0HVpvnGpQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UG2f_AnAz5ktiq6SRiFitKw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U7GEcqoI5hbtyTXUHRFC4OA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U7GEcqoI5hbtyTXUHRFC4OA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05168.DrugIgG.png" /></div></a><div id="U7GEcqoI5hbtyTXUHRFC4OA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05168.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U7GEcqoI5hbtyTXUHRFC4OA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UU8LsU_sT2S_lkotEcVc5YQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UpbdM7mtavho41X64vConEw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UpbdM7mtavho41X64vConEw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05164.DrugIgG.png" /></div></a><div id="UpbdM7mtavho41X64vConEw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05164.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UpbdM7mtavho41X64vConEw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UwkbYvqDXUDIOQPaCj2Vy6A" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UCJ33vd0gySnHHVFYZM7UHw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UCJ33vd0gySnHHVFYZM7UHw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05152.DrugIgG.png" /></div></a><div id="UCJ33vd0gySnHHVFYZM7UHw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05152.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UCJ33vd0gySnHHVFYZM7UHw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Ut6880H6w9QjqBY5YlFvYlg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UWeOwQky3Kb0_d7DgGWZRqg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UWeOwQky3Kb0_d7DgGWZRqg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04514.DrugIgG.png" /></div></a><div id="UWeOwQky3Kb0_d7DgGWZRqg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04514.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UWeOwQky3Kb0_d7DgGWZRqg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UA75Rn4cQHe7JSLWT1p7K4w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UAr9i-Wc99FFoFUvGqWjq5g');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UAr9i-Wc99FFoFUvGqWjq5g');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05169.DrugIgG.png" /></div></a><div id="UAr9i-Wc99FFoFUvGqWjq5g" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05169.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UAr9i-Wc99FFoFUvGqWjq5g" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UkNu0ElVacSX6id5LkWlWjg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UlRClQO5PojfUojQPQ5Agxg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UlRClQO5PojfUojQPQ5Agxg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04060.DrugIgG.png" /></div></a><div id="UlRClQO5PojfUojQPQ5Agxg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04060.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UlRClQO5PojfUojQPQ5Agxg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UNUtdbCj1pXOLr5JZbqbRag" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U-wwc7MF58kgkmCEG8N_8qw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U-wwc7MF58kgkmCEG8N_8qw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04650.DrugIgG.png" /></div></a><div id="U-wwc7MF58kgkmCEG8N_8qw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04650.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U-wwc7MF58kgkmCEG8N_8qw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UM2Ev1fwZkPphOablYV_3vw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UJ9lExMnI1NaNz2_7mOL3rA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UJ9lExMnI1NaNz2_7mOL3rA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05200.DrugIgG.png" /></div></a><div id="UJ9lExMnI1NaNz2_7mOL3rA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05200.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UJ9lExMnI1NaNz2_7mOL3rA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UVzWewBbHBYYyuH8XiTtGog" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_USNIBYm4bATu-Tz3CQqZvYQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('USNIBYm4bATu-Tz3CQqZvYQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04062.DrugIgG.png" /></div></a><div id="USNIBYm4bATu-Tz3CQqZvYQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04062.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_USNIBYm4bATu-Tz3CQqZvYQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UqDSWcV2ZD1S5fF9Vw9fo3w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U7zP8cryBVR7d90hFe7t7Hg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U7zP8cryBVR7d90hFe7t7Hg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05145.DrugIgG.png" /></div></a><div id="U7zP8cryBVR7d90hFe7t7Hg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05145.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U7zP8cryBVR7d90hFe7t7Hg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UF-uW_de47GqGx2RhgC4DcA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Uf7VvRZdH8zFSmjboVoEdiA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uf7VvRZdH8zFSmjboVoEdiA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04380.DrugIgG.png" /></div></a><div id="Uf7VvRZdH8zFSmjboVoEdiA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04380.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Uf7VvRZdH8zFSmjboVoEdiA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UVz6R0_BDm0dYjReyr29GFg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UY893MZVnS1Yl93u4U4t4FA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UY893MZVnS1Yl93u4U4t4FA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04668.DrugIgG.png" /></div></a><div id="UY893MZVnS1Yl93u4U4t4FA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04668.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UY893MZVnS1Yl93u4U4t4FA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UEGKp0meZvEikoPhotx9Qmg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U8AAr-CT0QuDp995VO70TQg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U8AAr-CT0QuDp995VO70TQg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05162.DrugIgG.png" /></div></a><div id="U8AAr-CT0QuDp995VO70TQg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05162.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U8AAr-CT0QuDp995VO70TQg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UO0dy4aEkVPkL7YeoeeNNtA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UhzzVqC1Plelly4RCQpwPZg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UhzzVqC1Plelly4RCQpwPZg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05161.DrugIgG.png" /></div></a><div id="UhzzVqC1Plelly4RCQpwPZg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05161.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UhzzVqC1Plelly4RCQpwPZg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UqZcrH56Rwc7fq37dI9jcig" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UXHEALXvcDGDC4dgpGQxzzQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UXHEALXvcDGDC4dgpGQxzzQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05203.DrugIgG.png" /></div></a><div id="UXHEALXvcDGDC4dgpGQxzzQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05203.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UXHEALXvcDGDC4dgpGQxzzQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U_N7TGEfmlfEdTIxR-VD1YA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UI11jACt9aPjVAm3uV0VMuw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UI11jACt9aPjVAm3uV0VMuw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04630.DrugIgG.png" /></div></a><div id="UI11jACt9aPjVAm3uV0VMuw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04630.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UI11jACt9aPjVAm3uV0VMuw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UsUqf8FJyHI-YevXYhifZig" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U0K5XkfgJbv_u59LF812qOg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U0K5XkfgJbv_u59LF812qOg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04151.DrugIgG.png" /></div></a><div id="U0K5XkfgJbv_u59LF812qOg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04151.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U0K5XkfgJbv_u59LF812qOg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UAxd2WbGMiKLYv508OZhF9Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UFQtSI9cp-mbe72yfY-O32w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UFQtSI9cp-mbe72yfY-O32w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05205.DrugIgG.png" /></div></a><div id="UFQtSI9cp-mbe72yfY-O32w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05205.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UFQtSI9cp-mbe72yfY-O32w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UL4JhuDLHdslx9bcp6Uai2g" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Uy4hyOt1J0VMD0wbZg9Nwug');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uy4hyOt1J0VMD0wbZg9Nwug');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05206.DrugIgG.png" /></div></a><div id="Uy4hyOt1J0VMD0wbZg9Nwug" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05206.DrugIgG.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Uy4hyOt1J0VMD0wbZg9Nwug" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                    
                      
                        <div id="U_MQV51xZuNaIOCbS3bAznQ" class="content"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Pathways:</p>
                              
                                <li class="tab-title active"><a href="#UrkqWOrQcNZ-Gq0q6SmefMw">HTLV-I infection</a></li>
                                <li class="tab-title"><a href="#UgtdkpEiRMWZwQZdajeAg7Q">Herpes simplex infection</a></li>
                                <li class="tab-title"><a href="#UBOfru4A1QjBbFtKMvD2ZYA">Tuberculosis</a></li>
                                <li class="tab-title"><a href="#U0WFLNRhksy8RxFHUV49nNQ">Cell adhesion molecules (CAMs)</a></li>
                                <li class="tab-title"><a href="#UzdfJLPEW66hq4L1MB6sA5g">Cytokine-cytokine receptor interaction</a></li>
                                <li class="tab-title"><a href="#UZtrBCXgcZ7KrVZ6FgStF0Q">Influenza A</a></li>
                                <li class="tab-title"><a href="#Ulh74O5Gs-dZqnly0qJtG1A">Natural killer cell mediated cytotoxicity</a></li>
                                <li class="tab-title"><a href="#U1aipgUS_pC3HIWmnhYCROA">Epstein-Barr virus infection</a></li>
                                <li class="tab-title"><a href="#UcObY0B-pe_Qgf_Bk7wyqrg">Pathways in cancer</a></li>
                                <li class="tab-title"><a href="#USnKawnHlZDiJkHh5HffBcg">Toxoplasmosis</a></li>
                                <li class="tab-title"><a href="#UbgKYbB1y3REubzEcEkWT7Q">Osteoclast differentiation</a></li>
                                <li class="tab-title"><a href="#UARnezMH2l0rKt4jl3klslg">Chemokine signaling pathway</a></li>
                                <li class="tab-title"><a href="#ULAvV9aYikGPHeSW0FDaCNA">TNF signaling pathway</a></li>
                                <li class="tab-title"><a href="#UKIhnNvQ7DwVBZNJq38Qrjg">PI3K-Akt signaling pathway</a></li>
                                <li class="tab-title"><a href="#UHgQrQZAB-6tRM1u9JNKr6Q">Measles</a></li>
                                <li class="tab-title"><a href="#U-RvOuNpmkXBosR8b1rKShQ">Proteoglycans in cancer</a></li>
                                <li class="tab-title"><a href="#U8hQttt7e3hot57isSzwQoQ">Hepatitis B</a></li>
                                <li class="tab-title"><a href="#Ujt-qcji-LZs1F7Goo4bn5A">Viral carcinogenesis</a></li>
                                <li class="tab-title"><a href="#UObK0lA5j1p_ZdxbZtaNdDg">Jak-STAT signaling pathway</a></li>
                                <li class="tab-title"><a href="#UcfEl0qETThT06eFJhptcYA">MicroRNAs in cancer</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="UrkqWOrQcNZ-Gq0q6SmefMw" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UScOgLrKJQksk2-BzYgbieg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UScOgLrKJQksk2-BzYgbieg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05166.DrugNMC2.png" /></div></a><div id="UScOgLrKJQksk2-BzYgbieg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05166.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UScOgLrKJQksk2-BzYgbieg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UgtdkpEiRMWZwQZdajeAg7Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UewQL-e0tCVwA5Ih2Ph_NrA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UewQL-e0tCVwA5Ih2Ph_NrA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05168.DrugNMC2.png" /></div></a><div id="UewQL-e0tCVwA5Ih2Ph_NrA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05168.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UewQL-e0tCVwA5Ih2Ph_NrA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UBOfru4A1QjBbFtKMvD2ZYA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U-ds5LtFl9-I6BKOh73T_yw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U-ds5LtFl9-I6BKOh73T_yw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05152.DrugNMC2.png" /></div></a><div id="U-ds5LtFl9-I6BKOh73T_yw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05152.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U-ds5LtFl9-I6BKOh73T_yw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U0WFLNRhksy8RxFHUV49nNQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U6WVfJIxgx8uykkTESTz56w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U6WVfJIxgx8uykkTESTz56w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04514.DrugNMC2.png" /></div></a><div id="U6WVfJIxgx8uykkTESTz56w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04514.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U6WVfJIxgx8uykkTESTz56w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UzdfJLPEW66hq4L1MB6sA5g" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UOcdlKtNc3_HOquho4KWpvw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UOcdlKtNc3_HOquho4KWpvw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04060.DrugNMC2.png" /></div></a><div id="UOcdlKtNc3_HOquho4KWpvw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04060.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UOcdlKtNc3_HOquho4KWpvw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UZtrBCXgcZ7KrVZ6FgStF0Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_Ui8Yc4JIkW4kHc-fVIaNAKQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Ui8Yc4JIkW4kHc-fVIaNAKQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05164.DrugNMC2.png" /></div></a><div id="Ui8Yc4JIkW4kHc-fVIaNAKQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05164.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_Ui8Yc4JIkW4kHc-fVIaNAKQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Ulh74O5Gs-dZqnly0qJtG1A" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UTvf_LFxMrJgStXR62Bxezg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UTvf_LFxMrJgStXR62Bxezg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04650.DrugNMC2.png" /></div></a><div id="UTvf_LFxMrJgStXR62Bxezg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04650.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UTvf_LFxMrJgStXR62Bxezg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U1aipgUS_pC3HIWmnhYCROA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UeVb4lu-A6uavgQwPJhY4pA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UeVb4lu-A6uavgQwPJhY4pA');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05169.DrugNMC2.png" /></div></a><div id="UeVb4lu-A6uavgQwPJhY4pA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05169.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UeVb4lu-A6uavgQwPJhY4pA" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UcObY0B-pe_Qgf_Bk7wyqrg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_ULbYWKYl_Qhg9qhbPHmIT0g');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('ULbYWKYl_Qhg9qhbPHmIT0g');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05200.DrugNMC2.png" /></div></a><div id="ULbYWKYl_Qhg9qhbPHmIT0g" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05200.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_ULbYWKYl_Qhg9qhbPHmIT0g" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="USnKawnHlZDiJkHh5HffBcg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UPzuWeiEwatCl9yjvppAAog');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UPzuWeiEwatCl9yjvppAAog');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05145.DrugNMC2.png" /></div></a><div id="UPzuWeiEwatCl9yjvppAAog" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05145.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UPzuWeiEwatCl9yjvppAAog" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UbgKYbB1y3REubzEcEkWT7Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U0Z4KGF8IbI2ZkyEN0b2LAQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U0Z4KGF8IbI2ZkyEN0b2LAQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04380.DrugNMC2.png" /></div></a><div id="U0Z4KGF8IbI2ZkyEN0b2LAQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04380.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U0Z4KGF8IbI2ZkyEN0b2LAQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UARnezMH2l0rKt4jl3klslg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UEdYBYGVJSngcpQnhy0DF5Q');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UEdYBYGVJSngcpQnhy0DF5Q');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04062.DrugNMC2.png" /></div></a><div id="UEdYBYGVJSngcpQnhy0DF5Q" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04062.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UEdYBYGVJSngcpQnhy0DF5Q" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="ULAvV9aYikGPHeSW0FDaCNA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UMUiUVxldke0F7YwZXOojLQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UMUiUVxldke0F7YwZXOojLQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04668.DrugNMC2.png" /></div></a><div id="UMUiUVxldke0F7YwZXOojLQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04668.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UMUiUVxldke0F7YwZXOojLQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UKIhnNvQ7DwVBZNJq38Qrjg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UwZrk3w9eUdj7LIFwIid9LQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UwZrk3w9eUdj7LIFwIid9LQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04151.DrugNMC2.png" /></div></a><div id="UwZrk3w9eUdj7LIFwIid9LQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04151.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UwZrk3w9eUdj7LIFwIid9LQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UHgQrQZAB-6tRM1u9JNKr6Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UzM3TASl20eKtGf0blBWdnw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UzM3TASl20eKtGf0blBWdnw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05162.DrugNMC2.png" /></div></a><div id="UzM3TASl20eKtGf0blBWdnw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05162.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UzM3TASl20eKtGf0blBWdnw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U-RvOuNpmkXBosR8b1rKShQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UF9AeZmNElwgQWFFTzklYSg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UF9AeZmNElwgQWFFTzklYSg');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05205.DrugNMC2.png" /></div></a><div id="UF9AeZmNElwgQWFFTzklYSg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05205.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UF9AeZmNElwgQWFFTzklYSg" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U8hQttt7e3hot57isSzwQoQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_U4OGtZjumyuVOKIdXRL3c4w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U4OGtZjumyuVOKIdXRL3c4w');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05161.DrugNMC2.png" /></div></a><div id="U4OGtZjumyuVOKIdXRL3c4w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05161.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_U4OGtZjumyuVOKIdXRL3c4w" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Ujt-qcji-LZs1F7Goo4bn5A" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UPjrFUF87cYyHPik8L2b6zw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UPjrFUF87cYyHPik8L2b6zw');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05203.DrugNMC2.png" /></div></a><div id="UPjrFUF87cYyHPik8L2b6zw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05203.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UPjrFUF87cYyHPik8L2b6zw" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UObK0lA5j1p_ZdxbZtaNdDg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UaTrI6z7EfvjBCN1WnPXJKQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UaTrI6z7EfvjBCN1WnPXJKQ');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa04630.DrugNMC2.png" /></div></a><div id="UaTrI6z7EfvjBCN1WnPXJKQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa04630.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UaTrI6z7EfvjBCN1WnPXJKQ" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UcfEl0qETThT06eFJhptcYA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b><br /><a href="#" onclick="openPopup('Ub0CDIpvEXBa7gb65IWKakQ_UFExV7u2KSePFX72wj61Q9g');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UFExV7u2KSePFX72wj61Q9g');return false;"><div class="small-12 columns"><image src="./results/Pathview/hsa05206.DrugNMC2.png" /></div></a><div id="UFExV7u2KSePFX72wj61Q9g" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/Pathview/hsa05206.DrugNMC2.png" style="width: 100%;" /></div></div><div id="Ub0CDIpvEXBa7gb65IWKakQ_UFExV7u2KSePFX72wj61Q9g" class="annotation_pop_up"><h5></h5><p>For each KEGG pathway, genes within the panel are mapped to the pathway and differential expression information is overlaid on the protein-based KEGG pathway image. Pathway nodes shown in white have no genes in the panel that map to them. Pathway nodes in grey have corresponding genes in the panel, however no significant differential expression is observed. Nodes in blue denote downregulation relative to the selected baseline, whereas nodes in orange denote upregulation relative to the selected baseline.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                  
                </div>
              
            </ul></div></div></div>
          
        
        
          
            <div id="UqWuHjesj-_-CPzfxWVp58Q" class="panelContent" style="display:none"><div class="row"><div class="panel"><p>Genes previously shown to be characteristic of various cell populations are used to measure these populations' abundance.
        QC plots explore the validity of each cell type's measurements, Summary plots explore the joint behavior of cell types, and Covariates plots compare cell type measurements to covariates.
        In addition to raw cell type abundance measurements, relative abundances measuring various contrasts between cell types are reported.</p></div></div><div class="row"><div class="panel"><ul class="small-block-grid-2 medium-block-grid-3">
              
                <dl class="button-group radius"><li><a class="small button active" href="#" data-dropdown="drop-UCej8QE8aElnIPoMjdHvf9A" title="QC">QC</a></li><li><a class="small button" href="#" data-dropdown="drop-UHKArn-Gt25gVXtASYJ6Xww" title="Summary">Summary</a></li><li><a class="small button" href="#" data-dropdown="drop-Uo1Mdo-I41UYqQ4itK9ryQg" title="Covariates">Covariates</a></li></dl><ul id="drop-UCej8QE8aElnIPoMjdHvf9A" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                  
                    <li class="active"><a menuindex="QC|QC" href="#UYKWd1ysW5lU6PX6kW2g5KQ">QC</a></li>
                  
                </ul><ul id="drop-UHKArn-Gt25gVXtASYJ6Xww" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                  
                    <li><a menuindex="Summary|Raw" href="#ULex0RwKLXEaPEhJkGAwk7w">Raw</a></li>
                  
                </ul><ul id="drop-Uo1Mdo-I41UYqQ4itK9ryQg" class="content f-dropdown" data-dropdown-content="" data-tab=""><!--Drop-down menu - This comment is a workaround for an apparent bug with empty elements not closing.-->
                  
                    <li><a menuindex="Covariates|Raw" href="#UuyqSJehbZCs5XzwDT8mFSg">Raw</a></li>
                  
                </ul><div class="tabs-content">
                
                  
                    
                      
                        <div id="UYKWd1ysW5lU6PX6kW2g5KQ" class="content active"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Choose a Cell Type:</p>
                              
                                <li class="tab-title active"><a href="#UX0cwH4N2F8YPHih5Cj7w8A">Summary</a></li>
                                <li class="tab-title"><a href="#U541CA4R5Msk87v2O42i0Ng">B-cells</a></li>
                                <li class="tab-title"><a href="#UpUyz-3kTq30zFJ-fjok95w">Exhausted CD8</a></li>
                                <li class="tab-title"><a href="#UgTI7SZT8t4VmYUUA52c1Mg">Macrophages</a></li>
                                <li class="tab-title"><a href="#UVsw8HK4cQVeuXqcL00RGqA">Mast cells</a></li>
                                <li class="tab-title"><a href="#UdFdBr31PmArME0iHKezM5w">Neutrophils</a></li>
                                <li class="tab-title"><a href="#U6Ivnb_Wexa8yNO3jxg98Qg">NK CD56dim cells</a></li>
                                <li class="tab-title"><a href="#UYrhJNBSyMPN4rtNylXs_cA">T-cells</a></li>
                                <li class="tab-title"><a href="#UJTqEcjIccG427OqIPH6cAA">Th1 cells</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="UX0cwH4N2F8YPHih5Cj7w8A" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Barplot of p-values across cell types<br /><a href="#" onclick="openPopup('U4-0HZkVRniYq8i4aa3mzHA_U4-0HZkVRniYq8i4aa3mzHA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('U4-0HZkVRniYq8i4aa3mzHA');return false;"><div class="small-12 columns"><image src="./results/cell types/barplot of p-values of cell types.png" /></div></a><div id="U4-0HZkVRniYq8i4aa3mzHA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/barplot of p-values of cell types.png" style="width: 100%;" /></div></div><div id="U4-0HZkVRniYq8i4aa3mzHA_U4-0HZkVRniYq8i4aa3mzHA" class="annotation_pop_up"><h5>Barplot of p-values across cell types</h5><p>Barplot of p-values across cell types. p-values are -log10 transformed. Bars above the solid black line indicate statistically significant cell types at an alpha level of 0.01. Bars above the dashed black line indicate statistically significant cell types at an alpha level of 0.001.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U541CA4R5Msk87v2O42i0Ng" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: B-cells<br /><a href="#" onclick="openPopup('UTMWGKwmy0bDf89lOJ0GChg_UTMWGKwmy0bDf89lOJ0GChg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UTMWGKwmy0bDf89lOJ0GChg');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - B-cells.png" /></div></a><div id="UTMWGKwmy0bDf89lOJ0GChg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - B-cells.png" style="width: 100%;" /></div></div><div id="UTMWGKwmy0bDf89lOJ0GChg_UTMWGKwmy0bDf89lOJ0GChg" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: B-cells</h5><p>Normalized expression of the genes characteristic of B-cells.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UpUyz-3kTq30zFJ-fjok95w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: Exhausted CD8<br /><a href="#" onclick="openPopup('UeNXLNPf6w5AgOW7NyuM2Pg_UeNXLNPf6w5AgOW7NyuM2Pg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UeNXLNPf6w5AgOW7NyuM2Pg');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - Exhausted CD8.png" /></div></a><div id="UeNXLNPf6w5AgOW7NyuM2Pg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - Exhausted CD8.png" style="width: 100%;" /></div></div><div id="UeNXLNPf6w5AgOW7NyuM2Pg_UeNXLNPf6w5AgOW7NyuM2Pg" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: Exhausted CD8</h5><p>Normalized expression of the genes characteristic of Exhausted CD8.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UgTI7SZT8t4VmYUUA52c1Mg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: Macrophages<br /><a href="#" onclick="openPopup('UyCT2VQwLONS3JmBL67WmMQ_UyCT2VQwLONS3JmBL67WmMQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UyCT2VQwLONS3JmBL67WmMQ');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - Macrophages.png" /></div></a><div id="UyCT2VQwLONS3JmBL67WmMQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - Macrophages.png" style="width: 100%;" /></div></div><div id="UyCT2VQwLONS3JmBL67WmMQ_UyCT2VQwLONS3JmBL67WmMQ" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: Macrophages</h5><p>Normalized expression of the genes characteristic of Macrophages.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UVsw8HK4cQVeuXqcL00RGqA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: Mast cells<br /><a href="#" onclick="openPopup('UhcpyW9cA3SWVoy77a-TCIQ_UhcpyW9cA3SWVoy77a-TCIQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UhcpyW9cA3SWVoy77a-TCIQ');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - Mast cells.png" /></div></a><div id="UhcpyW9cA3SWVoy77a-TCIQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - Mast cells.png" style="width: 100%;" /></div></div><div id="UhcpyW9cA3SWVoy77a-TCIQ_UhcpyW9cA3SWVoy77a-TCIQ" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: Mast cells</h5><p>Normalized expression of the genes characteristic of Mast cells.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UdFdBr31PmArME0iHKezM5w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: Neutrophils<br /><a href="#" onclick="openPopup('UW1uPi2wKxMETGmRWfukWkA_UW1uPi2wKxMETGmRWfukWkA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UW1uPi2wKxMETGmRWfukWkA');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - Neutrophils.png" /></div></a><div id="UW1uPi2wKxMETGmRWfukWkA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - Neutrophils.png" style="width: 100%;" /></div></div><div id="UW1uPi2wKxMETGmRWfukWkA_UW1uPi2wKxMETGmRWfukWkA" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: Neutrophils</h5><p>Normalized expression of the genes characteristic of Neutrophils.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U6Ivnb_Wexa8yNO3jxg98Qg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: NK CD56dim cells<br /><a href="#" onclick="openPopup('UnN5so8kqLuuu4wvnB50tIw_UnN5so8kqLuuu4wvnB50tIw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UnN5so8kqLuuu4wvnB50tIw');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - NK CD56dim cells.png" /></div></a><div id="UnN5so8kqLuuu4wvnB50tIw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - NK CD56dim cells.png" style="width: 100%;" /></div></div><div id="UnN5so8kqLuuu4wvnB50tIw_UnN5so8kqLuuu4wvnB50tIw" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: NK CD56dim cells</h5><p>Normalized expression of the genes characteristic of NK CD56dim cells.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UYrhJNBSyMPN4rtNylXs_cA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: T-cells<br /><a href="#" onclick="openPopup('UH9XYrxYggmo9CJzvKIZj-g_UH9XYrxYggmo9CJzvKIZj-g');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UH9XYrxYggmo9CJzvKIZj-g');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - T-cells.png" /></div></a><div id="UH9XYrxYggmo9CJzvKIZj-g" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - T-cells.png" style="width: 100%;" /></div></div><div id="UH9XYrxYggmo9CJzvKIZj-g_UH9XYrxYggmo9CJzvKIZj-g" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: T-cells</h5><p>Normalized expression of the genes characteristic of T-cells.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UJTqEcjIccG427OqIPH6cAA" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>QC Plot for Cell Type Measurements: Th1 cells<br /><a href="#" onclick="openPopup('UVhijNPgQwcBHPMqwGfTETg_UVhijNPgQwcBHPMqwGfTETg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UVhijNPgQwcBHPMqwGfTETg');return false;"><div class="small-12 columns"><image src="./results/cell types/QC for cell scores - Th1 cells.png" /></div></a><div id="UVhijNPgQwcBHPMqwGfTETg" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/QC for cell scores - Th1 cells.png" style="width: 100%;" /></div></div><div id="UVhijNPgQwcBHPMqwGfTETg_UVhijNPgQwcBHPMqwGfTETg" class="annotation_pop_up"><h5>QC Plot for Cell Type Measurements: Th1 cells</h5><p>Normalized expression of the genes characteristic of Th1 cells.  
                                                  If a cell type's characteristic genes are specific to the cell type and stably expressed within it, they will be strongly correlated with a slope of 1. Substantial departures from this pattern indicate noisier quantification of cell type abundance.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                  
                
                
                  
                    
                      
                        <div id="ULex0RwKLXEaPEhJkGAwk7w" class="content"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Choose a Cell Type:</p>
                              
                                <li class="tab-title active"><a href="#UG8Ocgw_IJzB2gagV7KjRuw">Summary</a></li>
                                <li class="tab-title"><a href="#UOzSzKxe7eHwkGYSdT-Vz5w">Macrophages</a></li>
                                <li class="tab-title"><a href="#UYfHN_J5HkSCqa_FOxfmzoQ">Exhausted CD8</a></li>
                                <li class="tab-title"><a href="#UtRd45B9ZHoPnimiYB4kQlw">B-cells</a></li>
                                <li class="tab-title"><a href="#UqMJ9ewyWu40bK81CcOEd8Q">Neutrophils</a></li>
                                <li class="tab-title"><a href="#U97a0TCJ1gMnHw7J4b2aBjw">T-cells</a></li>
                                <li class="tab-title"><a href="#UiJlM3Ic3DS9OvYPS2CKSfg">NK CD56dim cells</a></li>
                                <li class="tab-title"><a href="#UFsCV4A4LZt4Inz8tm3AQ-g">Mast cells</a></li>
                                <li class="tab-title"><a href="#UI61zw_k7J4EjuB-FsTgxfQ">Th1 cells</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="UG8Ocgw_IJzB2gagV7KjRuw" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1 medium-block-grid-2">
                                      
                                        <li><p><b>Heatmap of Raw cell type measurements<br /><a href="#" onclick="openPopup('URl0hvfUtLRFc9f09t_rWyA_UVVfjPB-x_QFG3aWN705JRA');return false;"><span class="label round">More Plot Information</span></a>  <a href="./results/cell types/cell type scores - raw.csv"><span class="label round">Download raw cell type measurements</span></a></b></p><a href="#" onclick="openImagePopup('UVVfjPB-x_QFG3aWN705JRA');return false;"><div class="small-9 columns"><image src="./results/cell types/heatmap of cell types scores - raw.png" /></div><div class="small-3 columns"><image src="./results/color legend.png" /></div></a><div id="UVVfjPB-x_QFG3aWN705JRA" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/heatmap of cell types scores - raw.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend.png" /></div></div><div id="URl0hvfUtLRFc9f09t_rWyA_UVVfjPB-x_QFG3aWN705JRA" class="annotation_pop_up"><h5>Heatmap of Raw cell type measurements</h5><p>Heatmap showing raw abundance of cell types. Orange indicates high abundance; blue indicates low abundance. Note that our methods do not support the conclusion that one cell type is more abundant than another.  Rather, they support the claim that a given cell type is more abundant in one sample than in another.</p></div></li>
                                        <li><p><b>Heatmap of Correlation Matrix of Raw cell type measurements<br /><a href="#" onclick="openPopup('Um9yj67wZZ5Kfa0DQcnhmjw_UcNADbTe48ZGBOHLZ8d9Qaw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UcNADbTe48ZGBOHLZ8d9Qaw');return false;"><div class="small-12 columns"><image src="./results/cell types/cell type scores correlation heatmap - raw.png" /></div></a><div id="UcNADbTe48ZGBOHLZ8d9Qaw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/cell type scores correlation heatmap - raw.png" style="width: 100%;" /></div></div><div id="Um9yj67wZZ5Kfa0DQcnhmjw_UcNADbTe48ZGBOHLZ8d9Qaw" class="annotation_pop_up"><h5>Heatmap of Correlation Matrix of Raw cell type measurements</h5><p>Heatmap showing the correlation matrix of raw cell abundance. Orange and blue indicate positive and negative correlation, respectively.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UOzSzKxe7eHwkGYSdT-Vz5w" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Macrophages measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UM8CsUOQ6-9NENYMUZy1LQw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UM8CsUOQ6-9NENYMUZy1LQw');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - Macrophages - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UM8CsUOQ6-9NENYMUZy1LQw" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - Macrophages - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UM8CsUOQ6-9NENYMUZy1LQw" class="annotation_pop_up"><h5>Macrophages measurements vs. other cell type measurements</h5><p>Each panel plots Macrophages against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UYfHN_J5HkSCqa_FOxfmzoQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Exhausted CD8 measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_ULwvfI7t-kBgFdVb9gHNVNQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('ULwvfI7t-kBgFdVb9gHNVNQ');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - Exhausted CD8 - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="ULwvfI7t-kBgFdVb9gHNVNQ" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - Exhausted CD8 - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_ULwvfI7t-kBgFdVb9gHNVNQ" class="annotation_pop_up"><h5>Exhausted CD8 measurements vs. other cell type measurements</h5><p>Each panel plots Exhausted CD8 against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UtRd45B9ZHoPnimiYB4kQlw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>B-cells measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UdqelXwwgbTUAn7_-FjMOJQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UdqelXwwgbTUAn7_-FjMOJQ');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - B-cells - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UdqelXwwgbTUAn7_-FjMOJQ" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - B-cells - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UdqelXwwgbTUAn7_-FjMOJQ" class="annotation_pop_up"><h5>B-cells measurements vs. other cell type measurements</h5><p>Each panel plots B-cells against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UqMJ9ewyWu40bK81CcOEd8Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Neutrophils measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UYlr4C2TRnGUr5EB2B6GMUw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UYlr4C2TRnGUr5EB2B6GMUw');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - Neutrophils - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UYlr4C2TRnGUr5EB2B6GMUw" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - Neutrophils - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UYlr4C2TRnGUr5EB2B6GMUw" class="annotation_pop_up"><h5>Neutrophils measurements vs. other cell type measurements</h5><p>Each panel plots Neutrophils against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U97a0TCJ1gMnHw7J4b2aBjw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>T-cells measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UClSYujnvwKCBUP7avACyOg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UClSYujnvwKCBUP7avACyOg');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - T-cells - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UClSYujnvwKCBUP7avACyOg" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - T-cells - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UClSYujnvwKCBUP7avACyOg" class="annotation_pop_up"><h5>T-cells measurements vs. other cell type measurements</h5><p>Each panel plots T-cells against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UiJlM3Ic3DS9OvYPS2CKSfg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>NK CD56dim cells measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UzK3hz_IXFlijJLVc9bOKqQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UzK3hz_IXFlijJLVc9bOKqQ');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - NK CD56dim cells - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UzK3hz_IXFlijJLVc9bOKqQ" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - NK CD56dim cells - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UzK3hz_IXFlijJLVc9bOKqQ" class="annotation_pop_up"><h5>NK CD56dim cells measurements vs. other cell type measurements</h5><p>Each panel plots NK CD56dim cells against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UFsCV4A4LZt4Inz8tm3AQ-g" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Mast cells measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UQ5gGW4jn7XO90a9yPrpwMg');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UQ5gGW4jn7XO90a9yPrpwMg');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - Mast cells - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UQ5gGW4jn7XO90a9yPrpwMg" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - Mast cells - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UQ5gGW4jn7XO90a9yPrpwMg" class="annotation_pop_up"><h5>Mast cells measurements vs. other cell type measurements</h5><p>Each panel plots Mast cells against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UI61zw_k7J4EjuB-FsTgxfQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Th1 cells measurements vs. other cell type measurements<br /><a href="#" onclick="openPopup('UMr2TLsqif2xB2ljgIST71A_UA8I2aY0mMyO0msh2oj04VQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UA8I2aY0mMyO0msh2oj04VQ');return false;"><div class="small-9 columns"><image src="./results/cell types/cell scores pairs plot - raw - Th1 cells - colored by Drug.png" /></div><div class="small-3 columns"><image src="./results/color legend - Drug.png" /></div></a><div id="UA8I2aY0mMyO0msh2oj04VQ" class="image_pop_up"><div class="image_pop_up_content small-10 columns"><img src="./results/cell types/cell scores pairs plot - raw - Th1 cells - colored by Drug.png" style="width: 100%;" /></div><div class="small-2 columns"><image src="./results/color legend - Drug.png" /></div></div><div id="UMr2TLsqif2xB2ljgIST71A_UA8I2aY0mMyO0msh2oj04VQ" class="annotation_pop_up"><h5>Th1 cells measurements vs. other cell type measurements</h5><p>Each panel plots Th1 cells against another raw abundance measurements.  Points are colored by Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                  
                
                
                  
                    
                      
                        <div id="UuyqSJehbZCs5XzwDT8mFSg" class="content"><div class="small-12 columns"><ul class="small-block-grid-1">
                          
                            <div class="small-2 columns"><ul class="tabs vertical" data-tab=""><p>Choose a Cell Type:</p>
                              
                                <li class="tab-title active"><a href="#UlCydPapCtinLcfNktD4Vww">Summary</a></li>
                                <li class="tab-title"><a href="#U-XX8tgibjKS7Io_8-ON4xw">Macrophages</a></li>
                                <li class="tab-title"><a href="#USoIKNAj3AtRMPYiv-kfutw">Exhausted CD8</a></li>
                                <li class="tab-title"><a href="#UnHunfjQtBb5Jo1leRdBA8Q">B-cells</a></li>
                                <li class="tab-title"><a href="#U_p6iH7rSoZ3sqdbHO9GWbQ">Neutrophils</a></li>
                                <li class="tab-title"><a href="#Uiyzz_GOK89pQmouvePVTrg">T-cells</a></li>
                                <li class="tab-title"><a href="#UZR4PeGrKsVtDMcwqAvaArw">NK CD56dim cells</a></li>
                                <li class="tab-title"><a href="#Uq_Dsd7yCdRQFqmNvwCGEfg">Mast cells</a></li>
                                <li class="tab-title"><a href="#Ud4MIcTYfBJODoFL5tXIY4A">Th1 cells</a></li>
                              
                            </ul></div><div class="tabs-content">
                              
                                
                                  
                                    <div id="UlCydPapCtinLcfNktD4Vww" class="content active"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Raw cell type measurements vs. Drug<br /><a href="#" onclick="openPopup('Uzh9UcYA90JGktijJ-3LaeA_UiwxHIokU4nssCQZwA1ropw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UiwxHIokU4nssCQZwA1ropw');return false;"><div class="small-12 columns"><image src="./results/cell types/trend plot of cell type scores vs. Drug - raw - legend.png" /></div></a><div id="UiwxHIokU4nssCQZwA1ropw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/trend plot of cell type scores vs. Drug - raw - legend.png" style="width: 100%;" /></div></div><div id="Uzh9UcYA90JGktijJ-3LaeA_UiwxHIokU4nssCQZwA1ropw" class="annotation_pop_up"><h5>Raw cell type measurements vs. Drug</h5><p>Plots the raw cell type abundance measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U-XX8tgibjKS7Io_8-ON4xw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Macrophages Measurement vs. Drug<br /><a href="#" onclick="openPopup('USNXgLnFy5WI1t_0pwjLu9Q_UDu9IbVInSembXieJBYPohQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UDu9IbVInSembXieJBYPohQ');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - Macrophages.png" /></div></a><div id="UDu9IbVInSembXieJBYPohQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - Macrophages.png" style="width: 100%;" /></div></div><div id="USNXgLnFy5WI1t_0pwjLu9Q_UDu9IbVInSembXieJBYPohQ" class="annotation_pop_up"><h5>Macrophages Measurement vs. Drug</h5><p>Plots Macrophages measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="USoIKNAj3AtRMPYiv-kfutw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Exhausted CD8 Measurement vs. Drug<br /><a href="#" onclick="openPopup('UEt9z4_cT1Adeq7bPulAxLw_UGXpJAKHBWRk08xieuvjgTw');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UGXpJAKHBWRk08xieuvjgTw');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - Exhausted CD8.png" /></div></a><div id="UGXpJAKHBWRk08xieuvjgTw" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - Exhausted CD8.png" style="width: 100%;" /></div></div><div id="UEt9z4_cT1Adeq7bPulAxLw_UGXpJAKHBWRk08xieuvjgTw" class="annotation_pop_up"><h5>Exhausted CD8 Measurement vs. Drug</h5><p>Plots Exhausted CD8 measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UnHunfjQtBb5Jo1leRdBA8Q" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>B-cells Measurement vs. Drug<br /><a href="#" onclick="openPopup('U9HC18ThvrAsaLe1ICT8deQ_Uzz-2Ow6UgIiuFw_Itk7N2w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uzz-2Ow6UgIiuFw_Itk7N2w');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - B-cells.png" /></div></a><div id="Uzz-2Ow6UgIiuFw_Itk7N2w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - B-cells.png" style="width: 100%;" /></div></div><div id="U9HC18ThvrAsaLe1ICT8deQ_Uzz-2Ow6UgIiuFw_Itk7N2w" class="annotation_pop_up"><h5>B-cells Measurement vs. Drug</h5><p>Plots B-cells measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="U_p6iH7rSoZ3sqdbHO9GWbQ" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Neutrophils Measurement vs. Drug<br /><a href="#" onclick="openPopup('Udrw_O32V8gJiX5HT6lN8dQ_Uo5BWHWgRfKyyci6KWaEaWQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Uo5BWHWgRfKyyci6KWaEaWQ');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - Neutrophils.png" /></div></a><div id="Uo5BWHWgRfKyyci6KWaEaWQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - Neutrophils.png" style="width: 100%;" /></div></div><div id="Udrw_O32V8gJiX5HT6lN8dQ_Uo5BWHWgRfKyyci6KWaEaWQ" class="annotation_pop_up"><h5>Neutrophils Measurement vs. Drug</h5><p>Plots Neutrophils measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Uiyzz_GOK89pQmouvePVTrg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>T-cells Measurement vs. Drug<br /><a href="#" onclick="openPopup('UHwCJ2eQHh7owlBaecZXl0Q_UMtYv2TRslwUcps8_W2Q30w');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UMtYv2TRslwUcps8_W2Q30w');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - T-cells.png" /></div></a><div id="UMtYv2TRslwUcps8_W2Q30w" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - T-cells.png" style="width: 100%;" /></div></div><div id="UHwCJ2eQHh7owlBaecZXl0Q_UMtYv2TRslwUcps8_W2Q30w" class="annotation_pop_up"><h5>T-cells Measurement vs. Drug</h5><p>Plots T-cells measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="UZR4PeGrKsVtDMcwqAvaArw" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>NK CD56dim cells Measurement vs. Drug<br /><a href="#" onclick="openPopup('UIHG9bnp8EKlIL1xKwxWxGw_Unfz0lPkNhjgQhtnh_1dTCA');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('Unfz0lPkNhjgQhtnh_1dTCA');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - NK CD56dim cells.png" /></div></a><div id="Unfz0lPkNhjgQhtnh_1dTCA" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - NK CD56dim cells.png" style="width: 100%;" /></div></div><div id="UIHG9bnp8EKlIL1xKwxWxGw_Unfz0lPkNhjgQhtnh_1dTCA" class="annotation_pop_up"><h5>NK CD56dim cells Measurement vs. Drug</h5><p>Plots NK CD56dim cells measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Uq_Dsd7yCdRQFqmNvwCGEfg" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Mast cells Measurement vs. Drug<br /><a href="#" onclick="openPopup('U0uewf77tNV63rHT8o_UIYg_UNdZOkwTK6rB9bkJnTkQybQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UNdZOkwTK6rB9bkJnTkQybQ');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - Mast cells.png" /></div></a><div id="UNdZOkwTK6rB9bkJnTkQybQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - Mast cells.png" style="width: 100%;" /></div></div><div id="U0uewf77tNV63rHT8o_UIYg_UNdZOkwTK6rB9bkJnTkQybQ" class="annotation_pop_up"><h5>Mast cells Measurement vs. Drug</h5><p>Plots Mast cells measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                                
                                  
                                    <div id="Ud4MIcTYfBJODoFL5tXIY4A" class="content"><div class="small-10 columns"><ul class="small-block-grid-1">
                                      
                                        <li><p><b>Th1 cells Measurement vs. Drug<br /><a href="#" onclick="openPopup('UE8BS41YuARpz70cRdr_ZVQ_UYKHLPVTamaaJNNxRFtlgyQ');return false;"><span class="label round">More Plot Information</span></a></b></p><a href="#" onclick="openImagePopup('UYKHLPVTamaaJNNxRFtlgyQ');return false;"><div class="small-12 columns"><image src="./results/cell types/Drug - raw - Th1 cells.png" /></div></a><div id="UYKHLPVTamaaJNNxRFtlgyQ" class="image_pop_up"><div class="image_pop_up_content small-12 columns"><img src="./results/cell types/Drug - raw - Th1 cells.png" style="width: 100%;" /></div></div><div id="UE8BS41YuARpz70cRdr_ZVQ_UYKHLPVTamaaJNNxRFtlgyQ" class="annotation_pop_up"><h5>Th1 cells Measurement vs. Drug</h5><p>Plots Th1 cells measurements against Drug.</p></div></li>
                                      
                                    </ul></div></div>
                                  
                                
                              
                            </div>
                          
                        </ul></div></div>
                      
                    
                  
                
              </div>
                
                
              
            </ul></div></div></div>
          
        
        
          
            <div id="UgJSZ36tfOj9Lv9WE-UAU3Q" class="panelContent" style="display:none"><div class="row"><div class="panel"><p>Summary of the options that were chosen when running this analysis.</p></div></div><div class="row"><div class="panel"><ul class="small-block-grid-25 medium-block-grid-26">
              
                <h5><b>Analysis Details</b></h5>
                <p> Analysis name: P0090_P0091 <br /> Location of results: C:\Users\Nathan\Dropbox\Company\Xsphera\@Analysis\Nanostring\2021.01.22-Nomacam-MetaAnalysis <br /> File type of plots: png <br /><br /></p>
                <h5><b>Sample Annotation Details</b></h5>
                <p> Unique sample identifier: File.Name <br />Covariate1: Patient; Variable Type: categorical; Reference Level: P0090<br />Covariate2: Disease; Variable Type: categorical; Reference Level: RCC<br />Covariate3: Drug; Variable Type: categorical; Reference Level: Media<br />Covariate4: Dosage; Variable Type: categorical; Reference Level: 0<br />Covariate5: Day; Variable Type: categorical; Reference Level: D0<br /><br /></p>
                <h5><b>Gene Annotation Details</b></h5>
                <p> Column defining probe set membership: Official.Gene.Name <br /> Probe annotation column giving cell type-specific probes: Cell.Type <br /><br /></p>
                <h5><b>Fusion Analysis Details</b></h5>
                <p> Perform fusion analysis: No <br /> p-value end probe threshold: 0.05 <br /> p-value junction probe threshold: 0.05 <br /><br /></p>
                <h5><b>SNV Analysis Details</b></h5>
                <p> Perform snv analysis: No <br /> Fold change threshold(s): 1, 2 <br /> p-value threshold(s): 0.01, 0.01 <br /> Reference samples:<br /> NA <br /> Use EM post processing: Yes <br /> Use debiasing post processing: Yes <br /><br /></p>
                <h5><b>Normalization Details</b></h5>
                <p> Perform normalization on: none <br /> Number of normalization probes: NA <br /><br /></p>
                <h5><b>Pathway Scoring Analysis Details</b></h5>
                <p> Perform pathway scoring analysis: TRUE <br /> Covariates to include in plots: Drug <br /><br /></p>
                <h5><b>Differential Expression Analysis Details</b></h5>
                <p> Perform differential expression testing: TRUE <br /> Predictors: Drug <br /> Confounders: none <br /> P-value adjustment: BY <br /> Run gene set analysis TRUE <br /><br /></p>
                <h5><b>Pathview  Details</b></h5>
                <p> Display results using Pathview: TRUE <br /> Color Pathview pathview plots by: Fold-change <br /> P-value threshold: 0.05 <br /> Additional KEGG IDs: none <br /><br /></p>
                <h5><b>Cell Type Profiling Details</b></h5>
                <p> Perform cell type profiling: TRUE <br /> Covariates to include in plots: Drug <br /> Use all cell type genes or dynamically select a subset:  <br /> P-value threshold for reporting a cell type's results: 1 <br /> Show results at the level of raw cell type abundance: TRUE <br /> Show results at the level of relative cell type abundance: TRUE <br /> Use default contrasts for calculating relative cell type abundance: TRUE <br /><br /></p>
                <h5><b>Probe Descriptive Analysis Details</b></h5>
                <p> Run Probe Descriptive analysis: FALSE <br /><br /></p>
                <h5><b>Related Analytes Analysis Details</b></h5>
                <p> Related Analytes Analysis: FALSE <br /><br /></p>
                <h5><b>Warnings</b></h5>
                <p>  Geneset information is discarded: the following error occurred<br /> Error in get.set.matrix(validated.prb.annot = validated.prb.annot, set.column.id = prb.set.column, : At least 1 set needs to have size equal or larger than5<br />     <br /> Warning: Geneset analysis will be not be conducted because no geneset annotation is provided<br /> <br />  <br /> <br />Cytotoxic cells, DC, CD45, Treg, CD8 T cells, NK cells cell types dropped from Cell Type Profiling as all probes were discarded due to signals below background threshold expression.<br /><br />Warning: the following cell types in the cell types contrast file were not found in the probe annotation file's cell type column:<br /> CD45, Cytotoxic cells, DC, NK cells, Treg, CD8 T cells <br /> Therefore the following contrasts have been discarded:<br /> Total.TILs<br />B.cells.vs..TILs<br />Cytotoxic.cells.vs..TILs<br />DC.vs..TILs<br />Exhausted.CD8.vs..TILs<br />Macrophages.vs..TILs<br />Mast.cells.vs..TILs<br />Neutrophils.vs..TILs<br />NK.CD56dim.cells.vs..TILs<br />NK.cells.vs..TILs<br />T.cells.vs..TILs<br />Th1.cells.vs..TILs<br />Treg.vs..TILs<br />CD8.T.cells.vs..TILs<br />CD8.vs..Exhasuted.CD8<br />CD8.vs..Treg<br />CD8.vs..T.cells <br /> Warning: no cell type contrasts remained with valid data.  Only raw cell type abundance will be reported. <br /> Warning:  Pathway scoring can not be completed as there are no valid probesets defined <br /> <br /><br /></p>
              
            </ul></div></div></div>
          
        
      
    </div>
    <div class="modal">
      <!--Loading div-->
    </div>
    <script>
          $(document).foundation();
        </script>
    <script>
          $(document).ready(function() {

          // Initialize all of the data tables
          $('table.display').dataTable( {
          "order": [],
          "aLengthMenu": [[5,10,15,20,25],[5,10,15,20,25]],
          "pageLength": 25
          } );

          // Hide share button if the inc file isn't there
          if (!shareButton) {
          $(".share-report").hide();
          }

          $(".image_pop_up").on("click", function() {
          $(".ui-dialog-content").dialog("close");
          });

          // Adjust the text on each active drop-down
          $(".f-dropdown li.active a").each(function( index ) {
          var newActive = $('[data-dropdown="' + $(this).closest('ul').attr('id') + '"]');

          if (newActive.attr('title') != $(this).text()) {
          newActive.text(newActive.attr('title') + ': ' + $(this).text().replace(newActive.attr('title'), ""));
          }
          else{
          newActive.text($(this).text());
          }

          });

          // Make top-bar behave like a tab controls - activate on click
          $('section.top-bar-section ul.right li').click(function() {
          if (!$(this).hasClass("share-report")) {
          $('section.top-bar-section ul.right li').removeClass('active');
          $(this).addClass("active");
          }
          });

          // Hide's the drop-down after it's clicked and add active to its 'parent' button
          $(".f-dropdown").on("click", function(){
          $(this).closest('ul').removeClass('open').css('left', '-9999px');
          });

          // When a drop-down's child is clicked, set its parent buttons text
          $(".f-dropdown li a").on("click", function(){

          // Sync the other drop-downs that have the same name...
          var matchingNodes = $('.f-dropdown li a[menuindex="' + $(this).attr('menuindex') + '"]');

          jQuery.each(matchingNodes, function(i, val) {
          var hrefText = val.href.substring(val.href.indexOf('#'));
          $(hrefText).siblings().removeClass('active')
          $(hrefText).addClass('active');

          var previousActive = $('*[data-dropdown="' + $(this).closest('ul').attr('id') + '"]').closest('dl').find('li a.active');
          previousActive.removeClass('active');
          previousActive.text(previousActive.attr('title'));

          var newActive = $('[data-dropdown="' + $(this).closest('ul').attr('id') + '"]');

          if (newActive.attr('title') != $(this).text()) {
          newActive.text(newActive.attr('title') + ': ' + $(this).text().replace(newActive.attr('title'), ""));
          }
          else{
          newActive.text($(this).text());
          }

          newActive.addClass('active');

          });
          });

          // Sync all sub-navs with the same name and activate the matching content
          $('dl.sub-nav li a').on('click', function(){

          var menuName = $(this).attr('menuindex').split("|")[0];

          $("dl.sub-nav li a[menuindex^='" + menuName + "|']").parent().removeClass('active');
          $("dl.sub-nav li a[menuindex^='" + menuName + "|']").parent().parent().siblings('div.tabs-content').children('div.content.active').removeClass('active');

          var matchingNodes = $('dl.sub-nav li a:contains(' + $(this).text() + ')');
          matchingNodes.parent().addClass('active');

          jQuery.each(matchingNodes, function(i, val) {
          var hrefText = $(this).attr('href');
          $(hrefText).addClass('active');
          });

          });

          });
        </script>
  </body>
</html>