<!DOCTYPE html>
    <html>
    <head>
        <meta charset="UTF-8">
        <title>References Notes&colon;</title>
        <style>
</style>
        
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/markdown.css">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/Microsoft/vscode/extensions/markdown-language-features/media/highlight.css">
<style>
            body {
                font-family: -apple-system, BlinkMacSystemFont, 'Segoe WPC', 'Segoe UI', system-ui, 'Ubuntu', 'Droid Sans', sans-serif;
                font-size: 14px;
                line-height: 1.6;
            }
        </style>
        <style>
.task-list-item { list-style-type: none; } .task-list-item-checkbox { margin-left: -20px; vertical-align: middle; }
</style>
        
        
        
    </head>
    <body class="vscode-body vscode-light">
        <ul>
<li><a href="#references-notes">References Notes:</a>
<ul>
<li><a href="#crispr-based-screens-uncover-determinants-of-immunotherapy-response-in-multiple-myeloma">CRISPR-based screens uncover determinants of immunotherapy response in multiple myeloma</a>
<ul>
<li><a href="#summary"><strong>Summary</strong></a></li>
<li><a href="#introduction"><strong>Introduction</strong></a></li>
<li><a href="#results"><strong>Results</strong></a></li>
<li><a href="#discussion"><strong>Discussion</strong></a></li>
</ul>
</li>
</ul>
</li>
</ul>
<h1 id="references-notes">References Notes:</h1>
<h2 id="crispr-based-screens-uncover-determinants-of-immunotherapy-response-in-multiple-myeloma">CRISPR-based screens uncover determinants of immunotherapy response in multiple myeloma</h2>
<p>Ramkumar P, Abarientos AB, Tian R, et al. CRISPR-based screens uncover determinants of immunotherapy response in multiple myeloma. Blood Adv. 2020;4(13):2899-2911. doi:10.1182/bloodadvances.2019001346</p>
<h3 id="summary"><strong>Summary</strong></h3>
<p>Cancer cells commonly develop resistance to immunotherapy by loss of antigen expression. Here, we use our CRISPR interference– and CRISPR activation–based functional genomics platform to systematically identify pathways controlling cell surface expression of the multiple myeloma immunotherapy antigen B-cellmaturation antigen (BCMA). We discovered that pharmacologic inhibition of HDAC7 and the Sec61 complex increased cell surface BCMA.</p>
<h3 id="introduction"><strong>Introduction</strong></h3>
<p>An important mechanism by which cancer cells can become resistant to different forms of immunotherapy
in the clinic is the downregulation or loss of the targeted antigen,6,7 also termed “<u>antigen escape</u>.&quot; Ongoing clinical trials using BCMA-targeted chimeric antigen receptor T cells (CAR-T cells) have reported antigen loss in some patients experiencing relapse,4,5 indicating that reduced cell surface
levels of BCMA may be an important mechanism of therapy resistance.  The current study used a CRISPR-interference/CRISPR-activation (CRISPRi/CRISPRa) functional genomics platform15,16 to systematically elucidate the mechanisms by which the cell surface expression of BCMA is controlled in MM cells and to test whether some of these mechanisms would be potential targets for combination therapy to enhance BCMA-directed immunotherapy</p>
<h3 id="results"><strong>Results</strong></h3>
<p><strong>Genome-wide CRISPR screens to identify genes controlling cell surface BCMA expression</strong> To identify novel genes or pathways regulating cell surface expression of BCMA in MM cell line, genome-wide CRISPRi and CRISPRa screens were performed in an MM cell line (<strong>Figure 1A</strong>). To select a suitable cell line for these primary screens, levels of BCMA expression on the RNA (<a href="http://www.keatslab.org">http://www.keatslab.org</a>) and cell surface protein levels for a panel of MM cell lines (<strong>Figure 1B</strong>) were compared. We found that AMO1 cells expressed moderate levels of cell surface BCMA.</p>
<p>Briefly, AMO1 cells expressing the CRISPRi and CRISPRa machinery were transduced with the pooled genome-wide sgRNA library. The cells were then stained for cell surface BCMA by using fluorescent tagged antibody and subjected to flow sorting into low and high BCMA populations. Frequencies of cells expressing each sgRNA were identified by using next-generation sequencing.</p>
<p>Knocking down BCMA itself or its transcription factor POU2AF124 resulted in significant downregulation of cell surface BCMA expression.</p>
<p>Furthermore, all of the subunits of the g secretase complex were among the top hits, and their knockdown resulted in a significant increase in cell surface BCMA. This finding was consistent with previous reports showing that <strong>g secretase complex</strong> cleaves membrane-bound BCMA to a soluble form (sBCMA).9,25 Moreover, several functional categories of genes regulating expression levels of BCMA were identified, including subunits of the <strong>Sec61 translocon complex</strong>, <strong>peroxisome biogenesis</strong>, <strong>proteasome subunits</strong>, and <strong>regulators of transcription</strong>.</p>
<p>Our CRISPRa genome-wide screen (Figure 1D; supplemental Table 2) identified genes in the Mucin family (MUC1 and MUC21) and several genes involved in transcriptional regulation (POU2AF1, CBFA2T3, MAML2, and RUNX3) that regulated surface BCMA.</p>
<p><strong>Validation of hit genes in a panel of MM cell lines</strong>
To test the generality of the findings, we decided to validate hit genes from the primary screen performed on the AMO1 cells. <u><em>Forty-one hits</em></u> were chosen, based both on strength of the phenotype from the CRISPRi screen and on their potential as therapeutic targets. As observed in the primary CRISPRi screen, knockdown of the subunits of <strong>g secretase subunits</strong> significantly increased cell surface BCMA (Figure 2A). Several of the novel factors from our primary screen were also validated in the larger panel of MM cells (<strong>Figure 2A; supplemental Tables 3 and 4</strong>), most notably the transcriptional regulator <strong>HDAC7</strong> and <strong>SEC61A1</strong>, a subunit of the SEC61 translocon. Knockdown of both factors increased cell surface levels of BCMA but not CD38.</p>
<p><strong>Class IIa HDAC inhibition upregulates BCMA transcription</strong> Our primary CRISPRi screen showed that knockdown of HDAC7, but not any other HDAC gene, upregulates BCMA expression (Figure 3A) HDAC7 belongs to the class II family of HDACs. We validated our screen findings by using TMP269, a pharmacologic inhibitor targeting class IIa HDACs. The results showed a twofold increase in BCMA transcript levels (Figure 3E), indicating that class II HDAC inhibition regulates transcription of BCMA. Furthermore, treatment of RPMI8226 cells with a combination of TMP269 and the g-secretase inhibitor RO4929097 revealed a further increase in BCMA surface expression levels, supporting the notion that they act through different mechanisms (Figure 3F). Furthermore, treatment with TMP269 in K562 cells, which do not express BCMA, did not lead to an increase in BCMA expression (supplemental Figure 2B-C). These results indicate the potential for using class IIa HDAC inhibition to increase expression of BCMA in plasma cells in the context of BCMA-targeted immunotherapy.</p>
<p><strong>Sec61 translocon regulates BCMA protein levels</strong> Most integral plasma membrane proteins are inserted into membranes via the Sec61 translocon. Surprisingly, our CRISPRi screen identified that knockdown of genes in the SEC61 pathway, such as <strong>SEC61A1, SEC61G, SSR1, and OSTC</strong>, resulted in an increase (rather than a decrease) in BCMA cell surface levels (Figure 1C). Conversely,
SEC61A1 knockdown resulted in a decrease in <strong>CD38</strong> cell surface levels.  We observed that treatment of MM cell lines with increasing concentrations of these compounds (SEC61 inhibitors) results in an up to fivefold dose-dependent increase in cell surface BCMA levels and a decrease in CD38 levels with minimal cytotoxicity.</p>
<p>To validate our findings in primary patient cells, bone marrow mononuclear cells derived from various patients with MM (supplemental Table 5) were treated with increasing concentrations of TMP269 and RO4929097 for 24 hours.  An up to 2.5-fold increase was observed in cell surface BCMA with class IIa HDAC inhibition and an up to eightfold increase with g-secretase inhibition (Figure 5).</p>
<p><u>Immunotherapy agents such as ADC are sensitive to changes in expression of target antigen.35 We found that BCMA knockdown reduced sensitivity of MM cell lines to a BCMA-targeting ADC (HDP101)21,36 (supplemental Figure 5A-C). Moreover, when cells were treated with either the g secretase inhibitor RO4929097 or the Sec61 inhibitor PS3061 at concentrations increasing cell surface levels of BCMA (supplemental Figure 5D) in combination with HDP101, increased efficacy of the BCMA-targeted ADC on the MM cell lines was observed (supplemental Figure 5E).</u></p>
<h3 id="discussion"><strong>Discussion</strong></h3>
<ul>
<li>The current study established that CRISPR-based screens enable a systematic and scalable approach to identifying mechanisms of antigen expression. Uncovering pathways that regulate BCMA transcription, translation and trafficking to the cell surface enabled us to propose effective treatment strategies in patients with low basal expression of BCMA or after relapse due to antigen loss.</li>
<li>Our findings also raise several mechanistic questions, which will be the subject of future studies. In particular, it is not clear how SEC61 inhibition results in increased cell surface levels of BCMA. SEC61 knockdown was previously shown to upregulate a subset of proteins.39</li>
<li>our results indicate HDAC7 as a potential combination
target for BCMA-targeted immunotherapy</li>
<li>Our study also identified genes in the sialic acid biosynthesis
pathways GALE and GNE, knockdown of which sensitized MM cells to CAR-T cells. Although our BCMA expression screen identified these genes to upregulate BCMA, the effect size of sensitizing the cells to CAR-T cells was much higher than the change in BCMA expression, indicating that there could be additional mechanisms by which this pathway modulates response to CAR-T cells.</li>
</ul>

    </body>
    </html>